<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
    <link rel="manifest" href="/site.webmanifest">
    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="theme-color" content="#ffffff">
    <title>Awesome Bioie (caufieldjh/awesome-bioie) Overview - Track Awesome List</title>
    <meta property="og:url" content="https://www.trackawesomelist.com/caufieldjh/awesome-bioie/readme/" />
    <meta property="og:type" content="summary" />
    <meta property="og:title" content="Awesome Bioie Overview" />
    <meta property="og:description" content="üß´ A curated list of resources relevant to doing Biomedical Information Extraction (including BioNLP)" />
    <meta property="og:site_name" content="Track Awesome List" />
    <style>
      main {
        max-width: 1024px;
        margin: 0 auto;
        padding: 0 0.5em;
      }
      :root,[data-color-mode=light][data-light-theme=light],[data-color-mode=dark][data-dark-theme=light]{--color-canvas-default-transparent:rgba(255,255,255,0);--color-prettylights-syntax-comment:#6e7781;--color-prettylights-syntax-constant:#0550ae;--color-prettylights-syntax-entity:#8250df;--color-prettylights-syntax-storage-modifier-import:#24292f;--color-prettylights-syntax-entity-tag:#116329;--color-prettylights-syntax-keyword:#cf222e;--color-prettylights-syntax-string:#0a3069;--color-prettylights-syntax-variable:#953800;--color-prettylights-syntax-string-regexp:#116329;--color-prettylights-syntax-constant-other-reference-link:#0a3069;--color-fg-default:#24292f;--color-fg-muted:#57606a;--color-canvas-default:#fff;--color-canvas-subtle:#f6f8fa;--color-border-default:#d0d7de;--color-border-muted:#d8dee4;--color-neutral-muted:rgba(175,184,193,.2);--color-accent-fg:#0969da;--color-accent-emphasis:#0969da;--color-danger-fg:#cf222e}@media (prefers-color-scheme:light){[data-color-mode=auto][data-light-theme=light]{--color-canvas-default-transparent:rgba(255,255,255,0);--color-prettylights-syntax-comment:#6e7781;--color-prettylights-syntax-constant:#0550ae;--color-prettylights-syntax-entity:#8250df;--color-prettylights-syntax-storage-modifier-import:#24292f;--color-prettylights-syntax-entity-tag:#116329;--color-prettylights-syntax-keyword:#cf222e;--color-prettylights-syntax-string:#0a3069;--color-prettylights-syntax-variable:#953800;--color-prettylights-syntax-string-regexp:#116329;--color-prettylights-syntax-constant-other-reference-link:#0a3069;--color-fg-default:#24292f;--color-fg-muted:#57606a;--color-canvas-default:#fff;--color-canvas-subtle:#f6f8fa;--color-border-default:#d0d7de;--color-border-muted:#d8dee4;--color-neutral-muted:rgba(175,184,193,.2);--color-accent-fg:#0969da;--color-accent-emphasis:#0969da;--color-danger-fg:#cf222e}}@media (prefers-color-scheme:dark){[data-color-mode=auto][data-dark-theme=light]{--color-canvas-default-transparent:rgba(255,255,255,0);--color-prettylights-syntax-comment:#6e7781;--color-prettylights-syntax-constant:#0550ae;--color-prettylights-syntax-entity:#8250df;--color-prettylights-syntax-storage-modifier-import:#24292f;--color-prettylights-syntax-entity-tag:#116329;--color-prettylights-syntax-keyword:#cf222e;--color-prettylights-syntax-string:#0a3069;--color-prettylights-syntax-variable:#953800;--color-prettylights-syntax-string-regexp:#116329;--color-prettylights-syntax-constant-other-reference-link:#0a3069;--color-fg-default:#24292f;--color-fg-muted:#57606a;--color-canvas-default:#fff;--color-canvas-subtle:#f6f8fa;--color-border-default:#d0d7de;--color-border-muted:#d8dee4;--color-neutral-muted:rgba(175,184,193,.2);--color-accent-fg:#0969da;--color-accent-emphasis:#0969da;--color-danger-fg:#cf222e}}[data-color-mode=light][data-light-theme=dark],[data-color-mode=dark][data-dark-theme=dark]{--color-canvas-default-transparent:rgba(13,17,23,0);--color-prettylights-syntax-comment:#8b949e;--color-prettylights-syntax-constant:#79c0ff;--color-prettylights-syntax-entity:#d2a8ff;--color-prettylights-syntax-storage-modifier-import:#c9d1d9;--color-prettylights-syntax-entity-tag:#7ee787;--color-prettylights-syntax-keyword:#ff7b72;--color-prettylights-syntax-string:#a5d6ff;--color-prettylights-syntax-variable:#ffa657;--color-prettylights-syntax-string-regexp:#7ee787;--color-prettylights-syntax-constant-other-reference-link:#a5d6ff;--color-fg-default:#c9d1d9;--color-fg-muted:#8b949e;--color-canvas-default:#0d1117;--color-canvas-subtle:#161b22;--color-border-default:#30363d;--color-border-muted:#21262d;--color-neutral-muted:rgba(110,118,129,.4);--color-accent-fg:#58a6ff;--color-accent-emphasis:#1f6feb;--color-danger-fg:#f85149}@media (prefers-color-scheme:light){[data-color-mode=auto][data-light-theme=dark]{--color-canvas-default-transparent:rgba(13,17,23,0);--color-prettylights-syntax-comment:#8b949e;--color-prettylights-syntax-constant:#79c0ff;--color-prettylights-syntax-entity:#d2a8ff;--color-prettylights-syntax-storage-modifier-import:#c9d1d9;--color-prettylights-syntax-entity-tag:#7ee787;--color-prettylights-syntax-keyword:#ff7b72;--color-prettylights-syntax-string:#a5d6ff;--color-prettylights-syntax-variable:#ffa657;--color-prettylights-syntax-string-regexp:#7ee787;--color-prettylights-syntax-constant-other-reference-link:#a5d6ff;--color-fg-default:#c9d1d9;--color-fg-muted:#8b949e;--color-canvas-default:#0d1117;--color-canvas-subtle:#161b22;--color-border-default:#30363d;--color-border-muted:#21262d;--color-neutral-muted:rgba(110,118,129,.4);--color-accent-fg:#58a6ff;--color-accent-emphasis:#1f6feb;--color-danger-fg:#f85149}}@media (prefers-color-scheme:dark){[data-color-mode=auto][data-dark-theme=dark]{--color-canvas-default-transparent:rgba(13,17,23,0);--color-prettylights-syntax-comment:#8b949e;--color-prettylights-syntax-constant:#79c0ff;--color-prettylights-syntax-entity:#d2a8ff;--color-prettylights-syntax-storage-modifier-import:#c9d1d9;--color-prettylights-syntax-entity-tag:#7ee787;--color-prettylights-syntax-keyword:#ff7b72;--color-prettylights-syntax-string:#a5d6ff;--color-prettylights-syntax-variable:#ffa657;--color-prettylights-syntax-string-regexp:#7ee787;--color-prettylights-syntax-constant-other-reference-link:#a5d6ff;--color-fg-default:#c9d1d9;--color-fg-muted:#8b949e;--color-canvas-default:#0d1117;--color-canvas-subtle:#161b22;--color-border-default:#30363d;--color-border-muted:#21262d;--color-neutral-muted:rgba(110,118,129,.4);--color-accent-fg:#58a6ff;--color-accent-emphasis:#1f6feb;--color-danger-fg:#f85149}}.markdown-body{word-wrap:break-word;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;font-size:16px;line-height:1.5}.markdown-body:before{content:"";display:table}.markdown-body:after{clear:both;content:"";display:table}.markdown-body>:first-child{margin-top:0!important}.markdown-body>:last-child{margin-bottom:0!important}.markdown-body a:not([href]){color:inherit;text-decoration:none}.markdown-body .absent{color:var(--color-danger-fg)}.markdown-body .anchor{float:left;margin-left:-20px;padding-right:4px;line-height:1}.markdown-body .anchor:focus{outline:none}.markdown-body p,.markdown-body blockquote,.markdown-body ul,.markdown-body ol,.markdown-body dl,.markdown-body table,.markdown-body pre,.markdown-body details{margin-top:0;margin-bottom:16px}.markdown-body hr{height:.25em;background-color:var(--color-border-default);border:0;margin:24px 0;padding:0}.markdown-body blockquote{color:var(--color-fg-muted);border-left:.25em solid var(--color-border-default);padding:0 1em}.markdown-body blockquote>:first-child{margin-top:0}.markdown-body blockquote>:last-child{margin-bottom:0}.markdown-body h1,.markdown-body h2,.markdown-body h3,.markdown-body h4,.markdown-body h5,.markdown-body h6{margin-top:24px;margin-bottom:16px;font-weight:600;line-height:1.25}.markdown-body h1 .octicon-link,.markdown-body h2 .octicon-link,.markdown-body h3 .octicon-link,.markdown-body h4 .octicon-link,.markdown-body h5 .octicon-link,.markdown-body h6 .octicon-link{color:var(--color-fg-default);vertical-align:middle;visibility:hidden}.markdown-body h1:hover .anchor,.markdown-body h2:hover .anchor,.markdown-body h3:hover .anchor,.markdown-body h4:hover .anchor,.markdown-body h5:hover .anchor,.markdown-body h6:hover .anchor{text-decoration:none}.markdown-body h1:hover .anchor .octicon-link,.markdown-body h2:hover .anchor .octicon-link,.markdown-body h3:hover .anchor .octicon-link,.markdown-body h4:hover .anchor .octicon-link,.markdown-body h5:hover .anchor .octicon-link,.markdown-body h6:hover .anchor .octicon-link{visibility:visible}.markdown-body h1 tt,.markdown-body h1 code,.markdown-body h2 tt,.markdown-body h2 code,.markdown-body h3 tt,.markdown-body h3 code,.markdown-body h4 tt,.markdown-body h4 code,.markdown-body h5 tt,.markdown-body h5 code,.markdown-body h6 tt,.markdown-body h6 code{font-size:inherit;padding:0 .2em}.markdown-body h1{border-bottom:1px solid var(--color-border-muted);padding-bottom:.3em;font-size:2em}.markdown-body h2{border-bottom:1px solid var(--color-border-muted);padding-bottom:.3em;font-size:1.5em}.markdown-body h3{font-size:1.25em}.markdown-body h4{font-size:1em}.markdown-body h5{font-size:.875em}.markdown-body h6{color:var(--color-fg-muted);font-size:.85em}.markdown-body summary h1,.markdown-body summary h2,.markdown-body summary h3,.markdown-body summary h4,.markdown-body summary h5,.markdown-body summary h6{display:inline-block}.markdown-body summary h1 .anchor,.markdown-body summary h2 .anchor,.markdown-body summary h3 .anchor,.markdown-body summary h4 .anchor,.markdown-body summary h5 .anchor,.markdown-body summary h6 .anchor{margin-left:-40px}.markdown-body summary h1,.markdown-body summary h2{border-bottom:0;padding-bottom:0}.markdown-body ul,.markdown-body ol{padding-left:2em}.markdown-body ul.no-list,.markdown-body ol.no-list{padding:0;list-style-type:none}.markdown-body ol[type="1"]{list-style-type:decimal}.markdown-body ol[type=a]{list-style-type:lower-alpha}.markdown-body ol[type=i]{list-style-type:lower-roman}.markdown-body div>ol:not([type]){list-style-type:decimal}.markdown-body ul ul,.markdown-body ul ol,.markdown-body ol ol,.markdown-body ol ul{margin-top:0;margin-bottom:0}.markdown-body li>p{margin-top:16px}.markdown-body li+li{margin-top:.25em}.markdown-body dl{padding:0}.markdown-body dl dt{margin-top:16px;padding:0;font-size:1em;font-style:italic;font-weight:600}.markdown-body dl dd{margin-bottom:16px;padding:0 16px}.markdown-body table{width:100%;width:-webkit-max-content;width:-webkit-max-content;width:max-content;max-width:100%;display:block;overflow:auto}.markdown-body table th{font-weight:600}.markdown-body table th,.markdown-body table td{border:1px solid var(--color-border-default);padding:6px 13px}.markdown-body table tr{background-color:var(--color-canvas-default);border-top:1px solid var(--color-border-muted)}.markdown-body table tr:nth-child(2n){background-color:var(--color-canvas-subtle)}.markdown-body table img{background-color:rgba(0,0,0,0)}.markdown-body img{max-width:100%;box-sizing:content-box;background-color:var(--color-canvas-default)}.markdown-body img[align=right]{padding-left:20px}.markdown-body img[align=left]{padding-right:20px}.markdown-body .emoji{max-width:none;vertical-align:text-top;background-color:rgba(0,0,0,0)}.markdown-body span.frame{display:block;overflow:hidden}.markdown-body span.frame>span{float:left;width:auto;border:1px solid var(--color-border-default);margin:13px 0 0;padding:7px;display:block;overflow:hidden}.markdown-body span.frame span img{float:left;display:block}.markdown-body span.frame span span{clear:both;color:var(--color-fg-default);padding:5px 0 0;display:block}.markdown-body span.align-center{clear:both;display:block;overflow:hidden}.markdown-body span.align-center>span{text-align:center;margin:13px auto 0;display:block;overflow:hidden}.markdown-body span.align-center span img{text-align:center;margin:0 auto}.markdown-body span.align-right{clear:both;display:block;overflow:hidden}.markdown-body span.align-right>span{text-align:right;margin:13px 0 0;display:block;overflow:hidden}.markdown-body span.align-right span img{text-align:right;margin:0}.markdown-body span.float-left{float:left;margin-right:13px;display:block;overflow:hidden}.markdown-body span.float-left span{margin:13px 0 0}.markdown-body span.float-right{float:right;margin-left:13px;display:block;overflow:hidden}.markdown-body span.float-right>span{text-align:right;margin:13px auto 0;display:block;overflow:hidden}.markdown-body code,.markdown-body tt{background-color:var(--color-neutral-muted);border-radius:6px;margin:0;padding:.2em .4em;font-size:85%}.markdown-body code br,.markdown-body tt br{display:none}.markdown-body del code{-webkit-text-decoration:inherit;-webkit-text-decoration:inherit;text-decoration:inherit}.markdown-body samp{font-size:85%}.markdown-body pre{word-wrap:normal}.markdown-body pre code{font-size:100%}.markdown-body pre>code{word-break:normal;white-space:pre;background:0 0;border:0;margin:0;padding:0}.markdown-body .highlight{margin-bottom:16px}.markdown-body .highlight pre{word-break:normal;margin-bottom:0}.markdown-body .highlight pre,.markdown-body pre{background-color:var(--color-canvas-subtle);border-radius:6px;padding:16px;font-size:85%;line-height:1.45;overflow:auto}.markdown-body pre code,.markdown-body pre tt{max-width:auto;line-height:inherit;word-wrap:normal;background-color:rgba(0,0,0,0);border:0;margin:0;padding:0;display:inline;overflow:visible}.markdown-body .csv-data td,.markdown-body .csv-data th{text-align:left;white-space:nowrap;padding:5px;font-size:12px;line-height:1;overflow:hidden}.markdown-body .csv-data .blob-num{text-align:right;background:var(--color-canvas-default);border:0;padding:10px 8px 9px}.markdown-body .csv-data tr{border-top:0}.markdown-body .csv-data th{background:var(--color-canvas-subtle);border-top:0;font-weight:600}.markdown-body [data-footnote-ref]:before{content:"["}.markdown-body [data-footnote-ref]:after{content:"]"}.markdown-body .footnotes{color:var(--color-fg-muted);border-top:1px solid var(--color-border-default);font-size:12px}.markdown-body .footnotes ol{padding-left:16px}.markdown-body .footnotes li{position:relative}.markdown-body .footnotes li:target:before{pointer-events:none;content:"";border:2px solid var(--color-accent-emphasis);border-radius:6px;position:absolute;top:-8px;bottom:-8px;left:-24px;right:-8px}.markdown-body .footnotes li:target{color:var(--color-fg-default)}.markdown-body .footnotes .data-footnote-backref g-emoji{font-family:monospace}.markdown-body{background-color:var(--color-canvas-default);color:var(--color-fg-default)}.markdown-body a{color:var(--color-accent-fg);text-decoration:none}.markdown-body a:hover{text-decoration:underline}.markdown-body iframe{background-color:#fff;border:0;margin-bottom:16px}.markdown-body svg.octicon{fill:currentColor}.markdown-body .anchor>.octicon{display:inline}.markdown-body .highlight .token.keyword,.gfm-highlight .token.keyword{color:var(--color-prettylights-syntax-keyword)}.markdown-body .highlight .token.tag .token.class-name,.markdown-body .highlight .token.tag .token.script .token.punctuation,.gfm-highlight .token.tag .token.class-name,.gfm-highlight .token.tag .token.script .token.punctuation{color:var(--color-prettylights-syntax-storage-modifier-import)}.markdown-body .highlight .token.operator,.markdown-body .highlight .token.number,.markdown-body .highlight .token.boolean,.markdown-body .highlight .token.tag .token.punctuation,.markdown-body .highlight .token.tag .token.script .token.script-punctuation,.markdown-body .highlight .token.tag .token.attr-name,.gfm-highlight .token.operator,.gfm-highlight .token.number,.gfm-highlight .token.boolean,.gfm-highlight .token.tag .token.punctuation,.gfm-highlight .token.tag .token.script .token.script-punctuation,.gfm-highlight .token.tag .token.attr-name{color:var(--color-prettylights-syntax-constant)}.markdown-body .highlight .token.function,.gfm-highlight .token.function{color:var(--color-prettylights-syntax-entity)}.markdown-body .highlight .token.string,.gfm-highlight .token.string{color:var(--color-prettylights-syntax-string)}.markdown-body .highlight .token.comment,.gfm-highlight .token.comment{color:var(--color-prettylights-syntax-comment)}.markdown-body .highlight .token.class-name,.gfm-highlight .token.class-name{color:var(--color-prettylights-syntax-variable)}.markdown-body .highlight .token.regex,.gfm-highlight .token.regex{color:var(--color-prettylights-syntax-string)}.markdown-body .highlight .token.regex .regex-delimiter,.gfm-highlight .token.regex .regex-delimiter{color:var(--color-prettylights-syntax-constant)}.markdown-body .highlight .token.tag .token.tag,.markdown-body .highlight .token.property,.gfm-highlight .token.tag .token.tag,.gfm-highlight .token.property{color:var(--color-prettylights-syntax-entity-tag)}
    </style>
  </head>
  <body>
    <main data-color-mode="light" data-light-theme="light" data-dark-theme="dark" class="markdown-body">
      <h1>Awesome Bioie Overview</h1>
<p>üß´ A curated list of resources relevant to doing Biomedical Information Extraction (including BioNLP)</p>
<p><a href="/">üè† Home</a><span> ¬∑ </span><a href="https://www.trackawesomelist.com/caufieldjh/awesome-bioie/rss.xml">üî• Feed</a><span> ¬∑ </span><a href="https://trackawesomelist.us17.list-manage.com/subscribe?u=d2f0117aa829c83a63ec63c2f&id=36a103854c">üìÆ Subscribe</a><span> ¬∑ </span><a href="https://github.com/sponsors/theowenyoung">‚ù§Ô∏è  Sponsor</a><span> ¬∑ </span><a href="https://github.com/caufieldjh/awesome-bioie">üò∫ caufieldjh/awesome-bioie</a><span> ¬∑ </span><span>‚≠ê 381</span><span> ¬∑ </span><span>üè∑Ô∏è Health and Social Science</span></p>
<p><span>[ </span><a href="/caufieldjh/awesome-bioie/">Daily</a><span> / </span><a href="/caufieldjh/awesome-bioie/week/">Weekly</a><span> / </span><span>Overview</span><span> ]</span></p>
<div>
    <img src="https://github.com/caufieldjh/awesome-bioie/raw/main/images/abie_head.png" alt="Awesome BioIE Logo" />
    <br />
    <a href="https://awesome.re">
        <img src="https://awesome.re/badge-flat2.svg" alt="Awesome" />
    </a>
    <br />
    How to extract information from unstructured biomedical data and text.
    <br />

</div>

<p>What is BioIE? It includes any effort to extract structured information from <em>unstructured</em> (or, at least inconsistently structured) biological, clinical, or other biomedical data. The data source is often some collection of text documents written in technical language. If the resulting information is verifiable and consistent across sources, we may then consider it <em>knowledge</em>. Extracting information and producing knowledge from bio data requires adaptations upon methods developed for other types of unstructured data.</p>
<p>BioIE has undergone massive changes since the introduction of language models like BERT and the more recently created Large Language Models (LLMs; e.g., GPT-3/4, LLAMA2/3, Gemini, etc).</p>
<p>Resources included here are preferentially those available at no monetary cost and limited license requirements. Methods and datasets should be publicly accessible and actively maintained.</p>
<p>See also <a href="https://github.com/keon/awesome-nlp" rel="noopener noreferrer">awesome-nlp</a>, <a href="https://github.com/raivivek/awesome-biology" rel="noopener noreferrer">awesome-biology</a> and <a href="https://github.com/danielecook/Awesome-Bioinformatics" rel="noopener noreferrer">Awesome-Bioinformatics</a>.</p>
<p><em>Please read the <a href="https://github.com/caufieldjh/awesome-bioie/blob/main/README.md/contributing.md" rel="noopener noreferrer">contribution guidelines</a> before contributing. Please add your favourite resource by raising a <a href="https://github.com/caufieldjh/awesome-bioie/pulls" rel="noopener noreferrer">pull request</a>.</em></p>
<h2 id="contents"><a class="anchor" aria-hidden="true" tabindex="-1" href="#contents"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Contents</h2><ul>
<li><a href="#research-overviews">Research Overviews</a></li>
<li><a href="#groups-active-in-the-field">Groups Active in the Field</a></li>
<li><a href="#organizations">Organizations</a></li>
<li><a href="#journals-and-events">Journals and Events</a><ul>
<li><a href="#journals">Journals</a></li>
<li><a href="#conferences-and-other-events">Conferences and Other Events</a></li>
<li><a href="#challenges">Challenges</a></li>
</ul>
</li>
<li><a href="#tutorials">Tutorials</a><ul>
<li><a href="#guides">Guides</a></li>
<li><a href="#video-lectures-and-online-courses">Video Lectures and Online Courses</a></li>
</ul>
</li>
<li><a href="#code-libraries">Code Libraries</a><ul>
<li><a href="#repos-for-specific-datasets">Repos for Specific Datasets</a></li>
</ul>
</li>
<li><a href="#tools-platforms-and-services">Tools, Platforms, and Services</a><ul>
<li><a href="#annotation-tools">Annotation Tools</a></li>
</ul>
</li>
<li><a href="#techniques-and-models">Techniques and Models</a></li>
<li><a href="#datasets">Datasets</a><ul>
<li><a href="#biomedical-text-sources">Biomedical Text Sources</a></li>
<li><a href="#annotated-text-data">Annotated Text Data</a></li>
<li><a href="#protein-protein-interaction-annotated-corpora">Protein-protein Interaction Annotated Corpora</a></li>
<li><a href="#other-datasets">Other Datasets</a></li>
</ul>
</li>
<li><a href="#ontologies-and-controlled-vocabularies">Ontologies and Controlled Vocabularies</a></li>
<li><a href="#data-models">Data Models</a></li>
<li><a href="#credits">Credits</a></li>
</ul>
<h2 id="research-overviews"><a class="anchor" aria-hidden="true" tabindex="-1" href="#research-overviews"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Research Overviews</h2><h3 id="llms-in-biomedical-ie"><a class="anchor" aria-hidden="true" tabindex="-1" href="#llms-in-biomedical-ie"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>LLMs in Biomedical IE</h3><ul>
<li><a href="http://dx.doi.org/10.1101/2024.04.24.24306315" rel="noopener noreferrer">Large language models in healthcare: A comprehensive benchmark</a> - a statistical and human evaluation of sixteen different LLMs applied to medical language tasks.</li>
<li><a href="https://doi.org/10.1186/s12911-024-02459-6" rel="noopener noreferrer">Assessing the research landscape and clinical utility of large language models: a scoping review</a> - a high-level review of LLM applications in medicine as of March 2024.</li>
<li><a href="https://doi.org/10.1016/s2589-7500(24)00061-x" rel="noopener noreferrer">Ethical and regulatory challenges of large language models in medicine</a> - a review of ethical issues arising from applications of LLMs in biomedicine.</li>
<li><a href="http://dx.doi.org/10.1145/3442188.3445922" rel="noopener noreferrer">On the Dangers of Stochastic Parrots: Can Language Models Be Too Big? ü¶ú</a> - a frequently referenced but still relevant work concerning the roles, applications, and risks of language models.</li>
</ul>
<h3 id="pre-llm-overviews"><a class="anchor" aria-hidden="true" tabindex="-1" href="#pre-llm-overviews"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Pre-LLM Overviews</h3><ul>
<li><a href="https://www.ahajournals.org/doi/full/10.1161/CIRCRESAHA.117.310967" rel="noopener noreferrer">Biomedical Informatics on the Cloud: A Treasure Hunt for Advancing Cardiovascular Medicine</a> - An overview of how BioIE and bioinformatics workflows can be applied to questions in cardiovascular health and medicine research.</li>
<li><a href="https://www.sciencedirect.com/science/article/pii/S1532046417302563" rel="noopener noreferrer">Clinical information extraction applications: A literature review</a> - A review of clinical IE papers published as of September 2016. From Mayo Clinic group (see below).</li>
<li><a href="https://www.sciencedirect.com/science/article/pii/S1532046417301909" rel="noopener noreferrer">Literature Based Discovery: Models, methods, and trends</a> - A review of Literature Based Discovery (LBD), or the philosophy that meaningful connections may be found between seemingly unrelated scientific literature.<ul>
<li>For some historical context on LBD, see papers by University of Chicago's Don Swanson and Neil Smalheiser, including <a href="https://www.jstor.org/stable/4307965" rel="noopener noreferrer"><em>Undiscovered Public Knowledge</em></a> (paywalled) and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5771422/" rel="noopener noreferrer"><em>Rediscovering Don Swanson: the Past, Present and Future of Literature-Based Discovery</em></a>.</li>
</ul>
</li>
<li><a href="https://arxiv.org/abs/1702.03222" rel="noopener noreferrer">Mining Electronic Health Records (EHRs): A Survey</a> - A review of the methods and philosophy behind mining electronic health records, including using them for adverse event detection. See Table 2 for a list of relevant papers as of mid-2017.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250990/" rel="noopener noreferrer">Capturing the Patient's Perspective: a Review of Advances in Natural Language Processing of Health-Related Text</a> - A 2017 review of natural language processing methods applied to information extraction in health records and social media text. An important note from this review: "One of the main challenges in the field is the availability of data that can be shared and which can be used by the community to push the development of methods based on comparable and reproducible studies".</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="groups-active-in-the-field"><a class="anchor" aria-hidden="true" tabindex="-1" href="#groups-active-in-the-field"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Groups Active in the Field</h2><ul>
<li><a href="http://www.childrenshospital.org/research/labs/natural-language-processing-laboratory" rel="noopener noreferrer">Boston Children's Hospital Natural Language Processing Laboratory</a> - Led by Dr. Guergana Savova, formerly at Mayo Clinic and the Apache cTAKES project.</li>
<li><a href="https://www.brown.edu/academics/medical/about-us/research/centers-institutes-and-programs/biomedical-informatics/" rel="noopener noreferrer">Brown Center for Biomedical Informatics</a> - Based at Brown University and directed by Dr. Neil Sarkar, whose research group works on topics in clinical NLP and IE.</li>
<li><a href="http://compbio.ucdenver.edu/Hunter_lab/CCP_website/index.html" rel="noopener noreferrer">Center for Computational Pharmacology NLP Group</a> - based at University of Colorado, Denver and led by Larry Hunter - <a href="https://github.com/UCDenver-ccp" rel="noopener noreferrer">see their GitHub repos here.</a></li>
<li>Groups at U.S. National Institutes of Health (NIH) / National Library of Medicine (NLM):<ul>
<li><a href="https://www.lhncbc.nlm.nih.gov/personnel/dina-demner-fushman" rel="noopener noreferrer">Demner-Fushman group at NLM</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/research/bionlp/" rel="noopener noreferrer">BioNLP group at NCBI</a> - Develops improvements to biomedical literature search and curation (e.g., through PubMed), led by Dr. Zhiyong Lu.</li>
</ul>
</li>
<li><a href="https://jensenlab.org/" rel="noopener noreferrer">JensenLab</a> - Based at the Novo Nordisk Foundation Center for Protein Research at the University of Copenhagen, Denmark.</li>
<li><a href="http://www.nactem.ac.uk/" rel="noopener noreferrer">National Centre for Text Mining (NaCTeM)</a> - Based at the University of Manchester and led by Prof. Sophia Ananiadou, NaCTeM is concerned with text mining in general but has a particular focus on biomedical applications.</li>
<li><a href="https://www.mayo.edu/research/departments-divisions/department-health-sciences-research/medical-informatics/projects" rel="noopener noreferrer">Mayo Clinic's clinical natural language processing program</a> - Several groups at Mayo Clinic have made major contributions to BioIE (for example, the Apache cTAKES platform) over the past 20 years.</li>
<li><a href="https://monarchinitiative.org/" rel="noopener noreferrer">Monarch Initiative</a> - A joint effort between groups at Oregon State University, Oregon Health &amp; Science University, Lawrence Berkeley National Lab, The Jackson Laboratory, and several others, seeking to "integrate biological information using semantics, and present it in a novel way, leveraging phenotypes to bridge the knowledge gap".</li>
<li><a href="https://turkunlp.org/" rel="noopener noreferrer">TurkuNLP</a> - Based at the University of Turku and concerned with NLP in general with a focus on BioNLP and clinical applications.</li>
<li><a href="https://sbmi.uth.edu/nlp/" rel="noopener noreferrer">UTHealth Houston Biomedical Natural Language Processing Lab</a> - Based in the University of Texas Health Science Center at Houston, School of Biomedical Informatics and led by Dr. Hua Xu.</li>
<li><a href="https://nlp.cs.vcu.edu/" rel="noopener noreferrer">VCU Natural Language Processing Lab</a> - Based at Virginia Commonwealth University and led by Dr. Bridget McInnes.</li>
<li><a href="http://zaklab.org" rel="noopener noreferrer">Zaklab</a> - Group led by Dr. Isaac Kohane at Harvard Medical School's Department of Biomedical Informatics (Dr. Kohane is also a steward of the n2c2 (formerly i2b2) datasets - see <a href="#datasets">Datasets</a> below).</li>
<li><a href="https://www.dbmi.columbia.edu/" rel="noopener noreferrer">Columbia University Department of Biomedical Informatics</a> - Led by Drs. George Hripcsak and No√©mie Elhadad.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="organizations"><a class="anchor" aria-hidden="true" tabindex="-1" href="#organizations"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Organizations</h2><ul>
<li><a href="https://www.amia.org/" rel="noopener noreferrer">AMIA</a> - Many‚Äîbut certainly not all‚Äîindividuals studying biomedical informatics are members of the American Medical Informatics Association. AMIA publishes a journal, JAMIA (see below).</li>
<li><a href="https://imia-medinfo.org/" rel="noopener noreferrer">IMIA</a> - The International Medical Informatics Association. Publishes the IMIA Yearbook of Medical Informatics.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="journals-and-events"><a class="anchor" aria-hidden="true" tabindex="-1" href="#journals-and-events"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Journals and Events</h2><p>The interdisciplinary nature of BioIE means researchers in this space may share their findings and tools in a variety of ways. They may publish papers in journals, as is common in the biomedical and life sciences. They may publish conference papers and, upon acceptance, give a poster and/or oral presentation at an event; this is common practice in computer science and engineering fields. Conference papers are often published in collections of proceedings. Preprint publication is an increasingly popular and institutionally-accepted way to publish findings as well. Surrounding these formal, written products are the ideas of <a href="https://en.wikipedia.org/wiki/Open_science" rel="noopener noreferrer">open science</a>, open data, and open source: the code, data, and software BioIE researchers develop are valuable resources to the community.</p>
<h3 id="journals"><a class="anchor" aria-hidden="true" tabindex="-1" href="#journals"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Journals</h3><p>For preprints, try <a href="https://arxiv.org" rel="noopener noreferrer">arXiv</a>, especially the subjects Computation and Language (cs.CL) and Information Retrieval (cs.IR); <a href="https://www.biorxiv.org/" rel="noopener noreferrer">bioRxiv</a>; or <a href="https://www.medrxiv.org/" rel="noopener noreferrer">medRxiv</a>, especially the Health Informatics subject area.</p>
<ul>
<li><a href="https://academic.oup.com/database" rel="noopener noreferrer">Database</a> - Its subtitle is "The Journal of Biological Databases and Curation". Open access.</li>
<li><a href="https://academic.oup.com/nar" rel="noopener noreferrer">NAR</a> - Nucleic Acids Research. Has a broad biomolecular focus but is particularly notable for its annual database issue.</li>
<li><a href="https://academic.oup.com/jamia" rel="noopener noreferrer">JAMIA</a> - The Journal of the American Medical Informatics Association. Concerns "articles in the areas of clinical care, clinical research, translational science, implementation science, imaging, education, consumer health, public health, and policy".</li>
<li><a href="https://www.sciencedirect.com/journal/journal-of-biomedical-informatics" rel="noopener noreferrer">JBI</a> - The Journal of Biomedical Informatics. Not open access by default, though it does have an open-access "X" version.</li>
<li><a href="https://www.nature.com/sdata/" rel="noopener noreferrer">Scientific Data</a> - An open-access Springer Nature journal publishing "descriptions of scientifically valuable datasets, and research that advances the sharing and reuse of scientific data".</li>
</ul>
<h3 id="conferences-and-other-events"><a class="anchor" aria-hidden="true" tabindex="-1" href="#conferences-and-other-events"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Conferences and Other Events</h3><ul>
<li><a href="http://acm-bcb.org/" rel="noopener noreferrer">ACM-BCB</a> - The ACM Conference on Bioinformatics, Computational Biology, and Health Informatics. Held annually since 2010.</li>
<li><a href="http://ieeebibm.org/BIBM2019/" rel="noopener noreferrer">BIBM</a> - The IEEE International Conference on Bioinformatics and Biomedicine.</li>
<li><a href="https://www.iscb.org/about-ismb" rel="noopener noreferrer">ISMB</a> - The International Conference on Intelligent Systems for Molecular Biology is an annual conference hosted by the International Society for Computational Biology since 1993. Much of its focus has concerned bioinformatics and computational biology without an explicit clinical focus, though it has included an increasing amount of text mining content (e.g., the 2019 meeting included a <a href="http://cosi.iscb.org/wiki/TextMining:Home" rel="noopener noreferrer">full-day special session on Text Mining for Biology and Healthcare</a>). The meeting is combined with that of the European Conference on Computational Biology (ECCB) on odd-numbered years.</li>
<li><a href="https://psb.stanford.edu/" rel="noopener noreferrer">PSB</a> - The Pacific Symposium on Biocomputing.</li>
</ul>
<h3 id="challenges"><a class="anchor" aria-hidden="true" tabindex="-1" href="#challenges"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Challenges</h3><p>Some events in BioIE are organized around formal tasks and challenges in which groups develop their own computational solutions, given a dataset.</p>
<ul>
<li><a href="http://bioasq.org/" rel="noopener noreferrer">BioASQ</a> - Challenges on biomedical semantic indexing and question answering. Challenges and workshops held annually since 2013.</li>
<li><a href="https://biocreative.bioinformatics.udel.edu/" rel="noopener noreferrer">BioCreAtIvE workshop</a> - These workshops have been organized since 2004, with  BioCreative VI happening February 2017 and the <a href="https://sites.google.com/view/ohnlp2018/home" rel="noopener noreferrer">BioCreative/OHNLP Challenge</a> held in 2018. See <a href="#datasets">Datasets</a> below.</li>
<li><a href="http://alt.qcri.org/semeval2020/" rel="noopener noreferrer">SemEval workshop</a> - Tasks and evaluations in computational semantic analysis. Tasks vary by year but frequently cover scientific and/or biomedical language, e.g. the <a href="https://competitions.codalab.org/competitions/19948" rel="noopener noreferrer">SemEval-2019 Task 12 on Toponym Resolution in Scientific Papers</a>.</li>
<li><a href="https://knowledge-learning.github.io/ehealthkd-2019/" rel="noopener noreferrer">eHealth-KD</a> - Challenges for encouraging "development of software technologies to automatically extract a large variety of knowledge from eHealth documents written in the Spanish Language". Previously held as part of <a href="http://www.sepln.org/workshops/tass/" rel="noopener noreferrer">TASS</a>, an annual workshop for semantic analysis in Spanish.</li>
<li><a href="https://www.synapse.org/#!Synapse:syn18405991/wiki/589657" rel="noopener noreferrer">EHR DREAM Challenge</a> - Held along with several other <a href="http://dreamchallenges.org/" rel="noopener noreferrer">more bioinformatics-focused challenges</a>, this challenge opened in October 2019 and focuses on using electronic health record data to predict patient mortality. Uses a synthetic data set rather than real EHR contents.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="tutorials"><a class="anchor" aria-hidden="true" tabindex="-1" href="#tutorials"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Tutorials</h2><p>The field changes rapidly enough that tutorials any older than a few years are missing crucial details. A few more recent educational resources are listed below. A good foundational understanding of text mining techniques is very helpful, as is some basic experience with the Python and or R languages. The best option may be to learn by doing.</p>
<h3 id="llm-guides"><a class="anchor" aria-hidden="true" tabindex="-1" href="#llm-guides"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>LLM Guides</h3><p><em>TBD - watch this space!</em></p>
<h3 id="pre-llm-guides-lectures-and-courses"><a class="anchor" aria-hidden="true" tabindex="-1" href="#pre-llm-guides-lectures-and-courses"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Pre-LLM Guides, Lectures, and Courses</h3><ul>
<li><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0040020" rel="noopener noreferrer">Getting Started in Text Mining</a> - A brief introduction to bio-text mining from Cohen and Hunter. More than ten years old but still quite relevant. See also an <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1702322/" rel="noopener noreferrer">earlier paper by the same authors</a>.</li>
<li><a href="https://link.springer.com/book/10.1007/978-1-4939-0709-0" rel="noopener noreferrer">Biomedical Literature Mining</a> - A (non-free) volume of Methods in Molecular Biology from 2014. Chapters covers introductory principles in text mining, applications in the biological sciences, and potential for use in clinical or medical safety scenarios.</li>
<li><a href="https://www.coursera.org/learn/mining-medical-data" rel="noopener noreferrer">Coursera - Foundations of mining non-structured medical data</a> - About three hours worth of video lectures on working with medical data of various types and structures, including text and image data. Appears fairly high-level and intended for beginners.</li>
<li><a href="https://jensenlab.org/training/textmining/" rel="noopener noreferrer">JensenLab text mining exercises</a></li>
<li><a href="https://www.bits.vib.be/training-list/111-bits/training/previous-trainings/183-text-mining" rel="noopener noreferrer">VIB text mining and curation training</a> - This training workshop happenened in 2013 but the slides are still online.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="code-libraries"><a class="anchor" aria-hidden="true" tabindex="-1" href="#code-libraries"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Code Libraries</h2><ul>
<li><a href="https://biopython.org/" rel="noopener noreferrer">Biopython</a> - <a href="http://dx.doi.org/10.1093/bioinformatics/btp163" rel="noopener noreferrer">paper</a> - <a href="https://github.com/biopython/biopython" rel="noopener noreferrer">code</a> - Python tools primarily intended for bioinformatics and computational molecular biology purposes, but also a convenient way to obtain data, including documents/abstracts from PubMed (see Chapter 9 of the documentation).</li>
<li><a href="https://github.com/kilicogluh/Bio-SCoRes" rel="noopener noreferrer">Bio-SCoRes</a> - <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0148538" rel="noopener noreferrer">paper</a> - A framework for biomedical coreference resolution.</li>
<li><a href="https://github.com/NLPatVCU/medaCy" rel="noopener noreferrer">medaCy</a> - A system for building predictive medical natural language processing models. Built on the <a href="https://spacy.io/" rel="noopener noreferrer">spaCy</a> framework.</li>
<li><a href="https://github.com/allenai/SciSpaCy" rel="noopener noreferrer">ScispaCy</a> - <a href="https://arxiv.org/abs/1902.07669" rel="noopener noreferrer">paper</a> - A version of the <a href="https://spacy.io/" rel="noopener noreferrer">spaCy</a> framework for scientific and biomedical documents.</li>
<li><a href="https://github.com/ropensci/rentrez" rel="noopener noreferrer">rentrez</a> - R utilities for accessing NCBI resources, including PubMed.</li>
<li><a href="https://medium.com/@kormilitzin/med7-clinical-information-extraction-system-in-python-and-spacy-5e6f68ab1c68" rel="noopener noreferrer">Med7</a> - <a href="https://arxiv.org/abs/2003.01271" rel="noopener noreferrer">paper</a> - <a href="https://github.com/kormilitzin/med7" rel="noopener noreferrer">code</a> - a Python package and model (for use with spaCy) for doing NER with medication-related concepts.</li>
</ul>
<h3 id="repos-for-specific-datasets"><a class="anchor" aria-hidden="true" tabindex="-1" href="#repos-for-specific-datasets"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Repos for Specific Datasets</h3><ul>
<li><a href="https://github.com/MIT-LCP/mimic-code" rel="noopener noreferrer">mimic-code</a> - Code associated with the MIMIC-III dataset (see below). Includes some helpful <a href="https://github.com/MIT-LCP/mimic-code/tree/master/tutorials" rel="noopener noreferrer">tutorials</a>.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="tools-platforms-and-services"><a class="anchor" aria-hidden="true" tabindex="-1" href="#tools-platforms-and-services"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Tools, Platforms, and Services</h2><ul>
<li><a href="https://ctakes.apache.org/" rel="noopener noreferrer">cTAKES</a> - <a href="https://academic.oup.com/jamia/article/17/5/507/830823" rel="noopener noreferrer">paper</a> - <a href="https://github.com/apache/ctakes" rel="noopener noreferrer">code</a> - A system for processing the text in electronic medical records. Widely used and open source.</li>
<li><a href="https://clamp.uth.edu/" rel="noopener noreferrer">CLAMP</a> - <a href="https://academic.oup.com/jamia/article/25/3/331/4657212" rel="noopener noreferrer">paper</a> - A natural language processing toolkit intended for use with the text in clinical reports. Check out their <a href="https://clamp.uth.edu/clampdemo.php" rel="noopener noreferrer">live demo</a> first to see what it does. Usable at no cost for academic research.</li>
<li><a href="https://github.com/DeepPhe/DeepPhe-Release" rel="noopener noreferrer">DeepPhe</a> - A system for processing documents describing cancer presentations. Based on cTAKES (see above).</li>
<li><a href="https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/dnorm/" rel="noopener noreferrer">DNorm</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3810844/" rel="noopener noreferrer">paper</a> - A method for disease normalization, i.e., linking mentions of disease names and acronyms to unique concept identifiers. Downloadable version includes the NCBI Disease Corpus and BC5CDR (see Annotated Text Data below).</li>
<li><a href="https://www.ncbi.nlm.nih.gov/research/pubtator/" rel="noopener noreferrer">PubTator Central</a> - <a href="https://academic.oup.com/nar/article/47/W1/W587/5494727" rel="noopener noreferrer">paper</a> - A web platform that identifies five different types of biomedical concepts in PubMed articles and PubMed Central full texts. The full annotation sets are downloadable (see <a href="#annotated-text-data">Annotated Text Data</a> below).</li>
<li><a href="https://github.com/jakelever/pubrunner" rel="noopener noreferrer">Pubrunner</a> - A framework for running text mining tools on the newest set(s) of documents from PubMed.</li>
<li><a href="https://github.com/CogStack/CogStack-SemEHR" rel="noopener noreferrer">SemEHR</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6019046/" rel="noopener noreferrer">paper</a> - an IE infrastructure for electronic health records (EHR). Built on the <a href="https://github.com/CogStack" rel="noopener noreferrer">CogStack project</a>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/taggerone/" rel="noopener noreferrer">TaggerOne</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018376/" rel="noopener noreferrer">paper</a> - Performs concept normalization (see also DNorm above). Can be trained for specific concept types and can perform NER independent of other normalization functions.</li>
<li><a href="https://github.com/nikolamilosevic86/TabInOut" rel="noopener noreferrer">TabInOut</a> - <a href="https://link.springer.com/article/10.1007/s10032-019-00317-0" rel="noopener noreferrer">paper</a> - a framework for IE from tables in the literature.</li>
</ul>
<h3 id="annotation-tools"><a class="anchor" aria-hidden="true" tabindex="-1" href="#annotation-tools"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Annotation Tools</h3><ul>
<li><a href="https://github.com/weitechen/anafora" rel="noopener noreferrer">Anafora</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5657237/" rel="noopener noreferrer">paper</a> - An annotation tool with adjudication and progress tracking features.</li>
<li><a href="https://brat.nlplab.org/" rel="noopener noreferrer">brat</a> - <a href="https://www.aclweb.org/anthology/E12-2021/" rel="noopener noreferrer">paper</a> - <a href="https://github.com/nlplab/brat" rel="noopener noreferrer">code</a> - The brat rapid annotation tool. Supports producing text annotations visually, through the browser. Not subject specific; appropriate for many annotation projects. Visualization is based on that of the <a href="https://github.com/nlplab/stav/" rel="noopener noreferrer"><em>stav</em> tool</a>.</li>
<li><a href="https://ohnlp.github.io/MedTator/" rel="noopener noreferrer">MedTator</a> - <a href="https://academic.oup.com/bioinformatics/article-abstract/38/6/1776/6496915" rel="noopener noreferrer">paper</a> - <a href="https://github.com/OHNLP/MedTator" rel="noopener noreferrer">code</a> - An annotation tool designed to have minimal dependencies.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="techniques-and-models"><a class="anchor" aria-hidden="true" tabindex="-1" href="#techniques-and-models"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Techniques and Models</h2><h3 id="large-language-models"><a class="anchor" aria-hidden="true" tabindex="-1" href="#large-language-models"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Large Language Models</h3><p><em>TBD - watch this space!</em></p>
<h3 id="bert-models"><a class="anchor" aria-hidden="true" tabindex="-1" href="#bert-models"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>BERT models</h3><ul>
<li><a href="https://github.com/naver/biobert-pretrained" rel="noopener noreferrer">BioBERT</a> - <a href="https://arxiv.org/abs/1901.08746" rel="noopener noreferrer">paper</a> - <a href="https://github.com/dmis-lab/biobert" rel="noopener noreferrer">code</a> - A PubMed and PubMed Central-trained version of the <a href="https://arxiv.org/abs/1810.04805" rel="noopener noreferrer">BERT language model</a>.</li>
<li>ClinicalBERT - Two language models trained on clinical text have similar names. Both are BERT models trained on the text of clinical notes from the MIMIC-III dataset.<ul>
<li><a href="https://github.com/EmilyAlsentzer/clinicalBERT" rel="noopener noreferrer">Alsentzer et al Clinical BERT</a> - <a href="https://www.aclweb.org/anthology/W19-1909/" rel="noopener noreferrer">paper</a></li>
<li><a href="https://github.com/kexinhuang12345/clinicalBERT" rel="noopener noreferrer">Huang et al ClinicalBERT</a> - <a href="https://arxiv.org/abs/1904.05342" rel="noopener noreferrer">paper</a></li>
</ul>
</li>
<li><a href="https://github.com/allenai/scibert" rel="noopener noreferrer">SciBERT</a> - <a href="https://arxiv.org/abs/1903.10676" rel="noopener noreferrer">paper</a> - A BERT model trained on &gt;1M papers from the Semantic Scholar database.</li>
<li><a href="https://github.com/ncbi-nlp/bluebert" rel="noopener noreferrer">BlueBERT</a> - <a href="https://arxiv.org/abs/1906.05474" rel="noopener noreferrer">paper</a> - A BERT model pre-trained on PubMed text and MIMIC-III notes.</li>
<li><a href="https://microsoft.github.io/BLURB/models.html" rel="noopener noreferrer">PubMedBERT</a> - <a href="https://arxiv.org/abs/2007.15779" rel="noopener noreferrer">paper</a> - A BERT model trained from scratch on PubMed, with versions trained on abstracts+full texts and on abstracts alone.</li>
</ul>
<h3 id="gpt-2-models"><a class="anchor" aria-hidden="true" tabindex="-1" href="#gpt-2-models"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>GPT-2 models</h3><ul>
<li><a href="https://github.com/microsoft/BioGPT" rel="noopener noreferrer">BioGPT</a> - <a href="https://doi.org/10.1093/bib/bbac409" rel="noopener noreferrer">paper</a> - A GPT-2 model pre-trained on 15 million PubMed abstracts, along with fine-tuned versions for several biomedical tasks.</li>
</ul>
<h3 id="other-models"><a class="anchor" aria-hidden="true" tabindex="-1" href="#other-models"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Other models</h3><ul>
<li><a href="https://github.com/zalandoresearch/flair/pull/519" rel="noopener noreferrer">Flair embeddings from PubMed</a> - A language model available through the Flair framework and embedding method. Trained over a 5% sample of PubMed abstracts until 2015, or &gt; 1.2 million abstracts in total.</li>
</ul>
<h3 id="text-embeddings"><a class="anchor" aria-hidden="true" tabindex="-1" href="#text-embeddings"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Text Embeddings</h3><ul>
<li><a href="https://www.sciencedirect.com/science/article/pii/S1532046418301825" rel="noopener noreferrer">This paper from Hongfang Liu's group at Mayo Clinic</a> demonstrates how text embeddings trained on biomedical or clinical text can, but don't always, perform better on biomedical natural language processing tasks. That being said, pre-trained embeddings may be appropriate for your needs, especially as training domain-specific embeddings can be computationally intensive.</li>
<li><a href="http://bioasq.org/news/bioasq-releases-continuous-space-word-vectors-obtained-applying-word2vec-pubmed-abstracts" rel="noopener noreferrer">BioASQword2vec</a> - <a href="http://bioasq.lip6.fr/info/BioASQword2vec/" rel="noopener noreferrer">paper</a> - Qord embeddings derived from biomedical text (&gt;10 million PubMed abstracts) using the popular <a href="https://code.google.com/archive/p/word2vec/" rel="noopener noreferrer">word2vec</a> tool.</li>
<li><a href="https://figshare.com/articles/Improving_Biomedical_Word_Embeddings_with_Subword_Information_and_MeSH_Ontology/6882647" rel="noopener noreferrer">BioWordVec</a> - <a href="https://www.nature.com/articles/s41597-019-0055-0" rel="noopener noreferrer">paper</a> - <a href="https://github.com/ncbi-nlp/BioWordVec" rel="noopener noreferrer">code</a> - Word embeddings derived from biomedical text (&gt;27 million PubMed titles and abstracts), including subword embedding model based on MeSH.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="datasets"><a class="anchor" aria-hidden="true" tabindex="-1" href="#datasets"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Datasets</h2><p>Some of the datasets listed below require a <a href="https://www.nlm.nih.gov/databases/umls.html#license_request" rel="noopener noreferrer">UMLS Terminology Services (UTS) account</a> to access. Please note that the license granted with the UTS account requires users to submit an annual report about their use of UMLS resources. This is less challenging than it sounds.</p>
<h3 id="biomedical-text-sources"><a class="anchor" aria-hidden="true" tabindex="-1" href="#biomedical-text-sources"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Biomedical Text Sources</h3><p>The following resources contain indexed text documents in the biomedical sciences.</p>
<ul>
<li><a href="http://davis.wpi.edu/xmdv/datasets/ohsumed.html" rel="noopener noreferrer">OHSUMED</a> - <a href="https://dl.acm.org/citation.cfm?id=188557" rel="noopener noreferrer">paper</a> - 348,566 MEDLINE entries (title and sometimes abstract) from between 1987 and 1991. Includes MeSH labels. Primarily of historical significance.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/" rel="noopener noreferrer">PubMed Central Open Access Subset</a> - A set of PubMed Central articles usable under licenses other than traditional copyright, though the exact licenses vary by publication and source. Articles are available as PDF and XML.</li>
<li><a href="https://github.com/allenai/cord19" rel="noopener noreferrer">CORD-19</a> - A corpus of scholarly manuscripts concerning COVID-19. Articles are primarily from PubMed Central and preprint servers, though the set also includes metadata on papers without full-text availability.</li>
</ul>
<h3 id="annotated-text-data"><a class="anchor" aria-hidden="true" tabindex="-1" href="#annotated-text-data"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Annotated Text Data</h3><ul>
<li><a href="https://bionlp.nlm.nih.gov/tac2017adversereactions/" rel="noopener noreferrer">SPL-ADR-200db</a> - <a href="https://www.nature.com/articles/sdata20181" rel="noopener noreferrer">paper</a> - A pilot dataset containing standardised information, and annotations of occurence in text, about ~5,000 known adverse reactions for 200 FDA-approved drugs.</li>
<li><a href="https://sourceforge.net/projects/biocreative/files/" rel="noopener noreferrer">BioCreAtIvE 1</a> - <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-6-S1-S1" rel="noopener noreferrer">paper</a> - 15,000 sentences (10,000 training and 5,000 test) annotated for protein and gene names. 1,000 full text biomedical research articles annotated with protein names and Gene Ontology terms.</li>
<li><a href="https://sourceforge.net/projects/biocreative/files/" rel="noopener noreferrer">BioCreAtIvE 2</a> - <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2008-9-s2-s1" rel="noopener noreferrer">paper</a> - 15,000 sentences (10,000 training and 5,000 test, different from the first corpus) annotated for protein and gene names. 542 abstracts linked to EntrezGene identifiers. A variety of research articles annotated for features of protein‚Äìprotein interactions.</li>
<li><a href="https://biocreative.bioinformatics.udel.edu/accounts/login/?next=/resources/corpora/biocreative-v-cdr-corpus/" rel="noopener noreferrer">BioCreAtIvE V CDR Task Corpus (BC5CDR)</a> - <a href="https://academic.oup.com/database/article/doi/10.1093/database/baw068/2630414" rel="noopener noreferrer">paper</a> - 1,500 articles (title and abstract) published in 2014 or later, annotated for 4,409 chemicals, 5,818 diseases and 3116 chemical‚Äìdisease interactions. Requires registration.</li>
<li><a href="https://biocreative.bioinformatics.udel.edu/resources/corpora/chemprot-corpus-biocreative-vi/#chemprot-corpus-biocreative-vi:downloads" rel="noopener noreferrer">BioCreative VI CHEMPROT Corpus</a> - <a href="https://pdfs.semanticscholar.org/eed7/81f498b563df5a9e8a241c67d63dd1d92ad5.pdf" rel="noopener noreferrer">paper</a> - &gt;2,400 articles annotated with chemical-protein interactions of a variety of relation types. Requires registration.</li>
<li><a href="https://github.com/UCDenver-ccp/CRAFT" rel="noopener noreferrer">CRAFT</a> - <a href="https://link.springer.com/chapter/10.1007/978-94-024-0881-2_53" rel="noopener noreferrer">paper</a> - 67 full-text biomedical articles annotated in a variety of ways, including for concepts and coreferences. Now on version 5, including annotations linking concepts to the MONDO disease ontology.</li>
<li><a href="https://portal.dbmi.hms.harvard.edu/projects/n2c2-nlp/" rel="noopener noreferrer">n2c2 (formerly i2b2) Data</a> - The Department of Biomedical Informatics (DBMI) at Harvard Medical School manages data for the National NLP Clinical Challenges and the Informatics for Integrating Biology and the Bedside challenges running since 2006. They require registration before access and use. Datasets include a variety of topics. See the <a href="https://portal.dbmi.hms.harvard.edu/data-challenges/" rel="noopener noreferrer">list of data challenges</a> for individual descriptions.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/CBBresearch/Dogan/DISEASE/" rel="noopener noreferrer">NCBI Disease Corpus</a> - <a href="https://www.sciencedirect.com/science/article/pii/S1532046413001974" rel="noopener noreferrer">paper</a> - A corpus of 793 biomedical abstracts annotated with names of diseases and related concepts from MeSH and <a href="https://omim.org/" rel="noopener noreferrer">OMIM</a>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/research/pubtator/" rel="noopener noreferrer">PubTator Central datasets</a> - <a href="https://academic.oup.com/nar/article/47/W1/W587/5494727" rel="noopener noreferrer">paper</a> - Accessible through a RESTful API or FTP download. Includes annotations for &gt;29 million abstracts and ‚àº3 million full text documents.</li>
<li><a href="https://wsd.nlm.nih.gov/" rel="noopener noreferrer">Word Sense Disambiguation (WSD)</a> - <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-223" rel="noopener noreferrer">paper</a> - 203 ambiguous words and 37,888 automatically extracted instances of their use in biomedical research publications. Requires UTS account.</li>
<li><a href="https://www.nlm.nih.gov/databases/download/CQC.html" rel="noopener noreferrer">Clinical Questions Collection</a> - also known as CQC or the Iowa collection, these are several thousand questions posed by physicians during office visits along with the associated answers.</li>
<li><a href="http://2013.bionlp-st.org/" rel="noopener noreferrer">BioNLP ST 2013 datasets</a> - data from six shared tasks, though some may not be easily accessible; try the CG task set (BioNLP2013CG) for extensive entity and event annotations.</li>
<li><a href="https://rgai.inf.u-szeged.hu/node/105" rel="noopener noreferrer">BioScope</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2586758/" rel="noopener noreferrer">paper</a> - a corpus of sentences from medical and biological documents, annotated for negation, speculation, and linguistic scope.</li>
<li><a href="https://ftp.ncbi.nlm.nih.gov/pub/lu/BioRED/" rel="noopener noreferrer">BioRED</a> - <a href="https://arxiv.org/abs/2204.04263" rel="noopener noreferrer">paper</a> - a set of &gt;6.5K biomedical relation annotations, plus labels for novel findings.</li>
</ul>
<h3 id="protein-protein-interaction-annotated-corpora"><a class="anchor" aria-hidden="true" tabindex="-1" href="#protein-protein-interaction-annotated-corpora"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Protein-protein Interaction Annotated Corpora</h3><p>Protein-protein interactions are abbreviated as PPI. The following sets are available in <a href="http://bioc.sourceforge.net/" rel="noopener noreferrer">BioC format</a>. The older sets (AIMed, BioInfer, HPRD50, IEPA, and LLL) are available courtesy of the <a href="http://corpora.informatik.hu-berlin.de" rel="noopener noreferrer">WBI corpora repository</a> and were originally derived from the original sets by a <a href="http://mars.cs.utu.fi/PPICorpora/" rel="noopener noreferrer">group at Turku University</a>.</p>
<ul>
<li><a href="http://corpora.informatik.hu-berlin.de/corpora/brat2bioc/aimed_bioc.xml.zip" rel="noopener noreferrer">AIMed</a> - <a href="https://www.ncbi.nlm.nih.gov/pubmed/15811782" rel="noopener noreferrer">paper</a> - 225 MEDLINE abstracts annotated for PPI.</li>
<li><a href="http://bioc.sourceforge.net/BioC-BioGRID.html" rel="noopener noreferrer">BioC-BioGRID</a> - <a href="https://academic.oup.com/database/article/doi/10.1093/database/baw147/2884890" rel="noopener noreferrer">paper</a> - 120 full text articles annotated for PPI and genetic interactions. Used in the BioCreative V BioC task.</li>
<li><a href="http://corpora.informatik.hu-berlin.de/corpora/brat2bioc/bioinfer_bioc.xml.zip" rel="noopener noreferrer">BioInfer</a> - <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-50" rel="noopener noreferrer">paper</a> - 1,100 sentences from biomedical research abstracts annotated for relationships (including PPI), named entities, and syntactic dependencies. <a href="http://mars.cs.utu.fi/BioInfer/" rel="noopener noreferrer">Additional information and download links are here.</a></li>
<li><a href="http://corpora.informatik.hu-berlin.de/corpora/brat2bioc/hprd50_bioc.xml.zip" rel="noopener noreferrer">HPRD50</a> - <a href="https://academic.oup.com/bioinformatics/article/23/3/365/236564" rel="noopener noreferrer">paper</a> - 50 scientific abstracts referenced by the Human Protein Reference Database, annotated for PPI.</li>
<li><a href="http://corpora.informatik.hu-berlin.de/corpora/brat2bioc/iepa_bioc.xml.zip" rel="noopener noreferrer">IEPA</a> - <a href="http://psb.stanford.edu/psb-online/proceedings/psb02/abstracts/p326.html" rel="noopener noreferrer">paper</a> - 486 sentences from biomedical research abstracts annotated for pairs of co-occurring chemicals, including proteins (hence, PPI annotations).</li>
<li><a href="http://corpora.informatik.hu-berlin.de/corpora/brat2bioc/lll_bioc.xml.zip" rel="noopener noreferrer">LLL</a> - <a href="https://www.semanticscholar.org/paper/Learning-Language-in-Logic-Genic-Interaction-Nedellec/0863a9d71955341b7e1a6a6877d44d4f0bb22671" rel="noopener noreferrer">paper</a> - 77 sentences from research articles about the bacterium <em>Bacillus subtilis</em>, annotated for protein‚Äìgene interactions (so, fairly close to PPI annotations). <a href="http://genome.jouy.inra.fr/texte/LLLchallenge/#task1" rel="noopener noreferrer">Additional information is here.</a></li>
</ul>
<h3 id="other-datasets"><a class="anchor" aria-hidden="true" tabindex="-1" href="#other-datasets"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Other Datasets</h3><ul>
<li><a href="http://cohd.io" rel="noopener noreferrer">Columbia Open Health Data</a> - <a href="https://www.nature.com/articles/sdata2018273" rel="noopener noreferrer">paper</a> - A database of prevalence and co-occurrence frequencies of conditions, drugs, procedures, and patient demographics extracted from electronic health records. Does not include original record text.</li>
<li><a href="https://ctdbase.org/" rel="noopener noreferrer">Comparative Toxicogenomics Database</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323936/" rel="noopener noreferrer">paper</a> - A database of manually curated associations between chemicals, gene products, phenotypes, diseases, and environmental exposures. Useful for assembling ontologies of the related concepts, such as types of chemicals.</li>
<li><a href="https://mimic.physionet.org/" rel="noopener noreferrer">MIMIC-III</a> - <a href="https://www.nature.com/articles/sdata201635" rel="noopener noreferrer">paper</a> - Deidentified health data from ~60,000 intensive care unit admissions. Requires completion of an online training course (CITI training) and acceptance of a data use agreement prior to use.</li>
<li><a href="https://physionet.org/content/mimic-cxr/2.0.0/" rel="noopener noreferrer">MIMIC-CXR</a> - The MIMIC Chest X-Ray database. Contains more than 377,000 radiographic images and accompanying free-text radiology reports. As with MIMIC-III, requires acceptance of a data use agreement.</li>
<li><a href="https://www.nlm.nih.gov/research/umls/licensedcontent/umlsknowledgesources.html" rel="noopener noreferrer">UMLS Knowledge Sources</a> - <a href="https://www.ncbi.nlm.nih.gov/books/NBK9676/" rel="noopener noreferrer">reference manual</a> - A large and comprehensive collection of biomedical terminology and identifiers, as well as accompanying tools and scripts. Depending on your purposes, the single file MRCONSO.RRF may be sufficient, as this file contains unique identifiers and names for all concepts in the UMLS Metathesaurus. See also the Ontologies and Controlled Vocabularies section below.</li>
<li><a href="https://mimic-iv.mit.edu/" rel="noopener noreferrer">MIMIC-IV</a> - An update to MIMIC-III's multimodal patient data, now covering more recent years of admissions, plus a new data structure, emergency department records, and links to MIMIC-CXR images.</li>
<li><a href="https://eicu-crd.mit.edu/" rel="noopener noreferrer">eICU Collaborative Research Database</a> - <a href="https://www.nature.com/articles/sdata2018178" rel="noopener noreferrer">paper</a> - a database of observations from more than 200 thousand intensive care unit admissions, with consistent structure. Requires registration, training course completion, and data use agreement.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="ontologies-and-controlled-vocabularies"><a class="anchor" aria-hidden="true" tabindex="-1" href="#ontologies-and-controlled-vocabularies"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Ontologies and Controlled Vocabularies</h2><ul>
<li><a href="http://www.disease-ontology.org/" rel="noopener noreferrer">Disease Ontology</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383880/" rel="noopener noreferrer">paper</a> - An ontology of human diseases. Has cross-links to MeSH, ICD, NCI Thesaurus, SNOMED, and OMIM. Public domain. Available on <a href="https://github.com/DiseaseOntology/HumanDiseaseOntology" rel="noopener noreferrer">GitHub</a> and on the <a href="http://www.obofoundry.org/ontology/doid.html" rel="noopener noreferrer">OBO Foundry</a>.</li>
<li><a href="https://www.nlm.nih.gov/research/umls/rxnorm/index.html" rel="noopener noreferrer">RxNorm</a> - <a href="https://academic.oup.com/jamia/article/18/4/441/734170" rel="noopener noreferrer">paper</a> - Normalized names for clinical drugs and drug packs, with combined ingredients, strengths, and form, and assigned types from the Semantic Network (see below). Released monthly.</li>
<li><a href="https://lexsrv3.nlm.nih.gov/Specialist/Summary/lexicon.html" rel="noopener noreferrer">SPECIALIST Lexicon</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2247735/" rel="noopener noreferrer">paper</a> - A general English lexicon that includes many biomedical terms. Updated yearly since 1994 and still updated as of 2019. Part of UMLS but does not require UTS account to download.</li>
<li><a href="https://www.nlm.nih.gov/research/umls/knowledge_sources/metathesaurus/index.html" rel="noopener noreferrer">UMLS Metathesaurus</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC308795/" rel="noopener noreferrer">paper</a> - Mappings between &gt;3.8 million concepts, 14 million concept names, and &gt;200 sources of biomedical vocabulary and identifiers. It's big. It may help to prepare a subset of the Metathesaurus with the <a href="https://www.nlm.nih.gov/research/umls/implementation_resources/metamorphosys/help.html" rel="noopener noreferrer">MetamorphoSys installation tool</a> but we're still talking about ~30 Gb of disk space required for the 2019 release. <a href="https://www.ncbi.nlm.nih.gov/books/NBK9684/" rel="noopener noreferrer">See the manual here</a>. Requires UTS account.</li>
<li><a href="https://semanticnetwork.nlm.nih.gov/" rel="noopener noreferrer">UMLS Semantic Network</a> - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447396/" rel="noopener noreferrer">paper</a> - Lists of 133 semantic types and 54 semantic relationships covering biomedical concepts and vocabulary. Is the Metathesaurus too complex for your needs? Try this. Does not require UTS account to download.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="data-models"><a class="anchor" aria-hidden="true" tabindex="-1" href="#data-models"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Data Models</h2><p>Do you need a <a href="https://en.wikipedia.org/wiki/Data_model" rel="noopener noreferrer">data model</a>? If you are working with biomedical data, then the answer is probably "Yes".</p>
<ul>
<li><a href="https://biolink.github.io/biolink-model/" rel="noopener noreferrer">Biolink</a> - <a href="https://github.com/biolink/biolink-model" rel="noopener noreferrer">code</a> - A data model of biological entities. Provided as a <a href="https://yaml.org/" rel="noopener noreferrer">YAML</a> file.</li>
<li><a href="http://wiki.biouml.org/index.php/BioUML" rel="noopener noreferrer">BioUML</a> - <a href="https://academic.oup.com/nar/article/47/W1/W225/5498754" rel="noopener noreferrer">paper</a> - An architecture for biomedical data analysis, integration, and visualization. Conceptually based on the visual modeling language <a href="https://www.uml.org/what-is-uml.htm" rel="noopener noreferrer">UML</a>.</li>
<li><a href="https://github.com/OHDSI/CommonDataModel" rel="noopener noreferrer">OMOP Common Data Model</a> - a standard for observational healthcare data.</li>
</ul>
<p><a href="#contents">Back to Top</a></p>
<h2 id="credits"><a class="anchor" aria-hidden="true" tabindex="-1" href="#credits"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Credits</h2><p><a href="https://github.com/caufieldjh/awesome-bioie/blob/main/README.md/./CREDITS.md" rel="noopener noreferrer">Credits</a> for curators and sources.</p>
<h2 id="license"><a class="anchor" aria-hidden="true" tabindex="-1" href="#license"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>License</h2><p><a href="https://creativecommons.org/publicdomain/zero/1.0" rel="noopener noreferrer"><img src="https://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg" alt="CC0" /></a></p>
<p><a href="https://github.com/caufieldjh/awesome-bioie/blob/main/README.md/./LICENSE" rel="noopener noreferrer">License</a></p>


    </main>
  </body>
</html>
