<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
    <link rel="manifest" href="/site.webmanifest">
    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="theme-color" content="#ffffff">
    <title>Awesome Bioinformatics (danielecook/Awesome-Bioinformatics) Overview - Track Awesome List</title>
    <meta property="og:url" content="https://www.trackawesomelist.com/danielecook/Awesome-Bioinformatics/readme/" />
    <meta property="og:type" content="summary" />
    <meta property="og:title" content="Awesome Bioinformatics Overview" />
    <meta property="og:description" content="A curated list of awesome Bioinformatics libraries and software." />
    <meta property="og:site_name" content="Track Awesome List" />
    <style>
      main {
        max-width: 1024px;
        margin: 0 auto;
        padding: 0 0.5em;
      }
      :root,[data-color-mode=light][data-light-theme=light],[data-color-mode=dark][data-dark-theme=light]{--color-canvas-default-transparent:rgba(255,255,255,0);--color-prettylights-syntax-comment:#6e7781;--color-prettylights-syntax-constant:#0550ae;--color-prettylights-syntax-entity:#8250df;--color-prettylights-syntax-storage-modifier-import:#24292f;--color-prettylights-syntax-entity-tag:#116329;--color-prettylights-syntax-keyword:#cf222e;--color-prettylights-syntax-string:#0a3069;--color-prettylights-syntax-variable:#953800;--color-prettylights-syntax-string-regexp:#116329;--color-prettylights-syntax-constant-other-reference-link:#0a3069;--color-fg-default:#24292f;--color-fg-muted:#57606a;--color-canvas-default:#fff;--color-canvas-subtle:#f6f8fa;--color-border-default:#d0d7de;--color-border-muted:#d8dee4;--color-neutral-muted:rgba(175,184,193,.2);--color-accent-fg:#0969da;--color-accent-emphasis:#0969da;--color-danger-fg:#cf222e}@media (prefers-color-scheme:light){[data-color-mode=auto][data-light-theme=light]{--color-canvas-default-transparent:rgba(255,255,255,0);--color-prettylights-syntax-comment:#6e7781;--color-prettylights-syntax-constant:#0550ae;--color-prettylights-syntax-entity:#8250df;--color-prettylights-syntax-storage-modifier-import:#24292f;--color-prettylights-syntax-entity-tag:#116329;--color-prettylights-syntax-keyword:#cf222e;--color-prettylights-syntax-string:#0a3069;--color-prettylights-syntax-variable:#953800;--color-prettylights-syntax-string-regexp:#116329;--color-prettylights-syntax-constant-other-reference-link:#0a3069;--color-fg-default:#24292f;--color-fg-muted:#57606a;--color-canvas-default:#fff;--color-canvas-subtle:#f6f8fa;--color-border-default:#d0d7de;--color-border-muted:#d8dee4;--color-neutral-muted:rgba(175,184,193,.2);--color-accent-fg:#0969da;--color-accent-emphasis:#0969da;--color-danger-fg:#cf222e}}@media (prefers-color-scheme:dark){[data-color-mode=auto][data-dark-theme=light]{--color-canvas-default-transparent:rgba(255,255,255,0);--color-prettylights-syntax-comment:#6e7781;--color-prettylights-syntax-constant:#0550ae;--color-prettylights-syntax-entity:#8250df;--color-prettylights-syntax-storage-modifier-import:#24292f;--color-prettylights-syntax-entity-tag:#116329;--color-prettylights-syntax-keyword:#cf222e;--color-prettylights-syntax-string:#0a3069;--color-prettylights-syntax-variable:#953800;--color-prettylights-syntax-string-regexp:#116329;--color-prettylights-syntax-constant-other-reference-link:#0a3069;--color-fg-default:#24292f;--color-fg-muted:#57606a;--color-canvas-default:#fff;--color-canvas-subtle:#f6f8fa;--color-border-default:#d0d7de;--color-border-muted:#d8dee4;--color-neutral-muted:rgba(175,184,193,.2);--color-accent-fg:#0969da;--color-accent-emphasis:#0969da;--color-danger-fg:#cf222e}}[data-color-mode=light][data-light-theme=dark],[data-color-mode=dark][data-dark-theme=dark]{--color-canvas-default-transparent:rgba(13,17,23,0);--color-prettylights-syntax-comment:#8b949e;--color-prettylights-syntax-constant:#79c0ff;--color-prettylights-syntax-entity:#d2a8ff;--color-prettylights-syntax-storage-modifier-import:#c9d1d9;--color-prettylights-syntax-entity-tag:#7ee787;--color-prettylights-syntax-keyword:#ff7b72;--color-prettylights-syntax-string:#a5d6ff;--color-prettylights-syntax-variable:#ffa657;--color-prettylights-syntax-string-regexp:#7ee787;--color-prettylights-syntax-constant-other-reference-link:#a5d6ff;--color-fg-default:#c9d1d9;--color-fg-muted:#8b949e;--color-canvas-default:#0d1117;--color-canvas-subtle:#161b22;--color-border-default:#30363d;--color-border-muted:#21262d;--color-neutral-muted:rgba(110,118,129,.4);--color-accent-fg:#58a6ff;--color-accent-emphasis:#1f6feb;--color-danger-fg:#f85149}@media (prefers-color-scheme:light){[data-color-mode=auto][data-light-theme=dark]{--color-canvas-default-transparent:rgba(13,17,23,0);--color-prettylights-syntax-comment:#8b949e;--color-prettylights-syntax-constant:#79c0ff;--color-prettylights-syntax-entity:#d2a8ff;--color-prettylights-syntax-storage-modifier-import:#c9d1d9;--color-prettylights-syntax-entity-tag:#7ee787;--color-prettylights-syntax-keyword:#ff7b72;--color-prettylights-syntax-string:#a5d6ff;--color-prettylights-syntax-variable:#ffa657;--color-prettylights-syntax-string-regexp:#7ee787;--color-prettylights-syntax-constant-other-reference-link:#a5d6ff;--color-fg-default:#c9d1d9;--color-fg-muted:#8b949e;--color-canvas-default:#0d1117;--color-canvas-subtle:#161b22;--color-border-default:#30363d;--color-border-muted:#21262d;--color-neutral-muted:rgba(110,118,129,.4);--color-accent-fg:#58a6ff;--color-accent-emphasis:#1f6feb;--color-danger-fg:#f85149}}@media (prefers-color-scheme:dark){[data-color-mode=auto][data-dark-theme=dark]{--color-canvas-default-transparent:rgba(13,17,23,0);--color-prettylights-syntax-comment:#8b949e;--color-prettylights-syntax-constant:#79c0ff;--color-prettylights-syntax-entity:#d2a8ff;--color-prettylights-syntax-storage-modifier-import:#c9d1d9;--color-prettylights-syntax-entity-tag:#7ee787;--color-prettylights-syntax-keyword:#ff7b72;--color-prettylights-syntax-string:#a5d6ff;--color-prettylights-syntax-variable:#ffa657;--color-prettylights-syntax-string-regexp:#7ee787;--color-prettylights-syntax-constant-other-reference-link:#a5d6ff;--color-fg-default:#c9d1d9;--color-fg-muted:#8b949e;--color-canvas-default:#0d1117;--color-canvas-subtle:#161b22;--color-border-default:#30363d;--color-border-muted:#21262d;--color-neutral-muted:rgba(110,118,129,.4);--color-accent-fg:#58a6ff;--color-accent-emphasis:#1f6feb;--color-danger-fg:#f85149}}.markdown-body{word-wrap:break-word;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;font-size:16px;line-height:1.5}.markdown-body:before{content:"";display:table}.markdown-body:after{clear:both;content:"";display:table}.markdown-body>:first-child{margin-top:0!important}.markdown-body>:last-child{margin-bottom:0!important}.markdown-body a:not([href]){color:inherit;text-decoration:none}.markdown-body .absent{color:var(--color-danger-fg)}.markdown-body .anchor{float:left;margin-left:-20px;padding-right:4px;line-height:1}.markdown-body .anchor:focus{outline:none}.markdown-body p,.markdown-body blockquote,.markdown-body ul,.markdown-body ol,.markdown-body dl,.markdown-body table,.markdown-body pre,.markdown-body details{margin-top:0;margin-bottom:16px}.markdown-body hr{height:.25em;background-color:var(--color-border-default);border:0;margin:24px 0;padding:0}.markdown-body blockquote{color:var(--color-fg-muted);border-left:.25em solid var(--color-border-default);padding:0 1em}.markdown-body blockquote>:first-child{margin-top:0}.markdown-body blockquote>:last-child{margin-bottom:0}.markdown-body h1,.markdown-body h2,.markdown-body h3,.markdown-body h4,.markdown-body h5,.markdown-body h6{margin-top:24px;margin-bottom:16px;font-weight:600;line-height:1.25}.markdown-body h1 .octicon-link,.markdown-body h2 .octicon-link,.markdown-body h3 .octicon-link,.markdown-body h4 .octicon-link,.markdown-body h5 .octicon-link,.markdown-body h6 .octicon-link{color:var(--color-fg-default);vertical-align:middle;visibility:hidden}.markdown-body h1:hover .anchor,.markdown-body h2:hover .anchor,.markdown-body h3:hover .anchor,.markdown-body h4:hover .anchor,.markdown-body h5:hover .anchor,.markdown-body h6:hover .anchor{text-decoration:none}.markdown-body h1:hover .anchor .octicon-link,.markdown-body h2:hover .anchor .octicon-link,.markdown-body h3:hover .anchor .octicon-link,.markdown-body h4:hover .anchor .octicon-link,.markdown-body h5:hover .anchor .octicon-link,.markdown-body h6:hover .anchor .octicon-link{visibility:visible}.markdown-body h1 tt,.markdown-body h1 code,.markdown-body h2 tt,.markdown-body h2 code,.markdown-body h3 tt,.markdown-body h3 code,.markdown-body h4 tt,.markdown-body h4 code,.markdown-body h5 tt,.markdown-body h5 code,.markdown-body h6 tt,.markdown-body h6 code{font-size:inherit;padding:0 .2em}.markdown-body h1{border-bottom:1px solid var(--color-border-muted);padding-bottom:.3em;font-size:2em}.markdown-body h2{border-bottom:1px solid var(--color-border-muted);padding-bottom:.3em;font-size:1.5em}.markdown-body h3{font-size:1.25em}.markdown-body h4{font-size:1em}.markdown-body h5{font-size:.875em}.markdown-body h6{color:var(--color-fg-muted);font-size:.85em}.markdown-body summary h1,.markdown-body summary h2,.markdown-body summary h3,.markdown-body summary h4,.markdown-body summary h5,.markdown-body summary h6{display:inline-block}.markdown-body summary h1 .anchor,.markdown-body summary h2 .anchor,.markdown-body summary h3 .anchor,.markdown-body summary h4 .anchor,.markdown-body summary h5 .anchor,.markdown-body summary h6 .anchor{margin-left:-40px}.markdown-body summary h1,.markdown-body summary h2{border-bottom:0;padding-bottom:0}.markdown-body ul,.markdown-body ol{padding-left:2em}.markdown-body ul.no-list,.markdown-body ol.no-list{padding:0;list-style-type:none}.markdown-body ol[type="1"]{list-style-type:decimal}.markdown-body ol[type=a]{list-style-type:lower-alpha}.markdown-body ol[type=i]{list-style-type:lower-roman}.markdown-body div>ol:not([type]){list-style-type:decimal}.markdown-body ul ul,.markdown-body ul ol,.markdown-body ol ol,.markdown-body ol ul{margin-top:0;margin-bottom:0}.markdown-body li>p{margin-top:16px}.markdown-body li+li{margin-top:.25em}.markdown-body dl{padding:0}.markdown-body dl dt{margin-top:16px;padding:0;font-size:1em;font-style:italic;font-weight:600}.markdown-body dl dd{margin-bottom:16px;padding:0 16px}.markdown-body table{width:100%;width:-webkit-max-content;width:-webkit-max-content;width:max-content;max-width:100%;display:block;overflow:auto}.markdown-body table th{font-weight:600}.markdown-body table th,.markdown-body table td{border:1px solid var(--color-border-default);padding:6px 13px}.markdown-body table tr{background-color:var(--color-canvas-default);border-top:1px solid var(--color-border-muted)}.markdown-body table tr:nth-child(2n){background-color:var(--color-canvas-subtle)}.markdown-body table img{background-color:rgba(0,0,0,0)}.markdown-body img{max-width:100%;box-sizing:content-box;background-color:var(--color-canvas-default)}.markdown-body img[align=right]{padding-left:20px}.markdown-body img[align=left]{padding-right:20px}.markdown-body .emoji{max-width:none;vertical-align:text-top;background-color:rgba(0,0,0,0)}.markdown-body span.frame{display:block;overflow:hidden}.markdown-body span.frame>span{float:left;width:auto;border:1px solid var(--color-border-default);margin:13px 0 0;padding:7px;display:block;overflow:hidden}.markdown-body span.frame span img{float:left;display:block}.markdown-body span.frame span span{clear:both;color:var(--color-fg-default);padding:5px 0 0;display:block}.markdown-body span.align-center{clear:both;display:block;overflow:hidden}.markdown-body span.align-center>span{text-align:center;margin:13px auto 0;display:block;overflow:hidden}.markdown-body span.align-center span img{text-align:center;margin:0 auto}.markdown-body span.align-right{clear:both;display:block;overflow:hidden}.markdown-body span.align-right>span{text-align:right;margin:13px 0 0;display:block;overflow:hidden}.markdown-body span.align-right span img{text-align:right;margin:0}.markdown-body span.float-left{float:left;margin-right:13px;display:block;overflow:hidden}.markdown-body span.float-left span{margin:13px 0 0}.markdown-body span.float-right{float:right;margin-left:13px;display:block;overflow:hidden}.markdown-body span.float-right>span{text-align:right;margin:13px auto 0;display:block;overflow:hidden}.markdown-body code,.markdown-body tt{background-color:var(--color-neutral-muted);border-radius:6px;margin:0;padding:.2em .4em;font-size:85%}.markdown-body code br,.markdown-body tt br{display:none}.markdown-body del code{-webkit-text-decoration:inherit;-webkit-text-decoration:inherit;text-decoration:inherit}.markdown-body samp{font-size:85%}.markdown-body pre{word-wrap:normal}.markdown-body pre code{font-size:100%}.markdown-body pre>code{word-break:normal;white-space:pre;background:0 0;border:0;margin:0;padding:0}.markdown-body .highlight{margin-bottom:16px}.markdown-body .highlight pre{word-break:normal;margin-bottom:0}.markdown-body .highlight pre,.markdown-body pre{background-color:var(--color-canvas-subtle);border-radius:6px;padding:16px;font-size:85%;line-height:1.45;overflow:auto}.markdown-body pre code,.markdown-body pre tt{max-width:auto;line-height:inherit;word-wrap:normal;background-color:rgba(0,0,0,0);border:0;margin:0;padding:0;display:inline;overflow:visible}.markdown-body .csv-data td,.markdown-body .csv-data th{text-align:left;white-space:nowrap;padding:5px;font-size:12px;line-height:1;overflow:hidden}.markdown-body .csv-data .blob-num{text-align:right;background:var(--color-canvas-default);border:0;padding:10px 8px 9px}.markdown-body .csv-data tr{border-top:0}.markdown-body .csv-data th{background:var(--color-canvas-subtle);border-top:0;font-weight:600}.markdown-body [data-footnote-ref]:before{content:"["}.markdown-body [data-footnote-ref]:after{content:"]"}.markdown-body .footnotes{color:var(--color-fg-muted);border-top:1px solid var(--color-border-default);font-size:12px}.markdown-body .footnotes ol{padding-left:16px}.markdown-body .footnotes li{position:relative}.markdown-body .footnotes li:target:before{pointer-events:none;content:"";border:2px solid var(--color-accent-emphasis);border-radius:6px;position:absolute;top:-8px;bottom:-8px;left:-24px;right:-8px}.markdown-body .footnotes li:target{color:var(--color-fg-default)}.markdown-body .footnotes .data-footnote-backref g-emoji{font-family:monospace}.markdown-body{background-color:var(--color-canvas-default);color:var(--color-fg-default)}.markdown-body a{color:var(--color-accent-fg);text-decoration:none}.markdown-body a:hover{text-decoration:underline}.markdown-body iframe{background-color:#fff;border:0;margin-bottom:16px}.markdown-body svg.octicon{fill:currentColor}.markdown-body .anchor>.octicon{display:inline}.markdown-body .highlight .token.keyword,.gfm-highlight .token.keyword{color:var(--color-prettylights-syntax-keyword)}.markdown-body .highlight .token.tag .token.class-name,.markdown-body .highlight .token.tag .token.script .token.punctuation,.gfm-highlight .token.tag .token.class-name,.gfm-highlight .token.tag .token.script .token.punctuation{color:var(--color-prettylights-syntax-storage-modifier-import)}.markdown-body .highlight .token.operator,.markdown-body .highlight .token.number,.markdown-body .highlight .token.boolean,.markdown-body .highlight .token.tag .token.punctuation,.markdown-body .highlight .token.tag .token.script .token.script-punctuation,.markdown-body .highlight .token.tag .token.attr-name,.gfm-highlight .token.operator,.gfm-highlight .token.number,.gfm-highlight .token.boolean,.gfm-highlight .token.tag .token.punctuation,.gfm-highlight .token.tag .token.script .token.script-punctuation,.gfm-highlight .token.tag .token.attr-name{color:var(--color-prettylights-syntax-constant)}.markdown-body .highlight .token.function,.gfm-highlight .token.function{color:var(--color-prettylights-syntax-entity)}.markdown-body .highlight .token.string,.gfm-highlight .token.string{color:var(--color-prettylights-syntax-string)}.markdown-body .highlight .token.comment,.gfm-highlight .token.comment{color:var(--color-prettylights-syntax-comment)}.markdown-body .highlight .token.class-name,.gfm-highlight .token.class-name{color:var(--color-prettylights-syntax-variable)}.markdown-body .highlight .token.regex,.gfm-highlight .token.regex{color:var(--color-prettylights-syntax-string)}.markdown-body .highlight .token.regex .regex-delimiter,.gfm-highlight .token.regex .regex-delimiter{color:var(--color-prettylights-syntax-constant)}.markdown-body .highlight .token.tag .token.tag,.markdown-body .highlight .token.property,.gfm-highlight .token.tag .token.tag,.gfm-highlight .token.property{color:var(--color-prettylights-syntax-entity-tag)}
    </style>
  </head>
  <body>
    <main data-color-mode="light" data-light-theme="light" data-dark-theme="dark" class="markdown-body">
      <h1>Awesome Bioinformatics Overview</h1>
<p>A curated list of awesome Bioinformatics libraries and software.</p>
<p><a href="/">🏠 Home</a><span> · </span><a href="https://www.trackawesomelist.com/danielecook/Awesome-Bioinformatics/rss.xml">🔥 Feed</a><span> · </span><a href="https://trackawesomelist.us17.list-manage.com/subscribe?u=d2f0117aa829c83a63ec63c2f&id=36a103854c">📮 Subscribe</a><span> · </span><a href="https://github.com/sponsors/theowenyoung">❤️  Sponsor</a><span> · </span><a href="https://github.com/danielecook/Awesome-Bioinformatics">😺 danielecook/Awesome-Bioinformatics</a><span> · </span><span>⭐ 3.5K</span><span> · </span><span>🏷️ Miscellaneous</span></p>
<p><span>[ </span><a href="/danielecook/Awesome-Bioinformatics/">Daily</a><span> / </span><a href="/danielecook/Awesome-Bioinformatics/week/">Weekly</a><span> / </span><span>Overview</span><span> ]</span></p>
<h1 id="awesome-bioinformatics-awesome-check-urls-generate-toc"><a class="anchor" aria-hidden="true" tabindex="-1" href="#awesome-bioinformatics-awesome-check-urls-generate-toc"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Awesome Bioinformatics <a href="https://github.com/sindresorhus/awesome" rel="noopener noreferrer"><img src="https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg" alt="Awesome" /></a> <a href="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/url-check.yml" rel="noopener noreferrer"><img src="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/url-check.yml/badge.svg" alt="Check URLs" /></a> <a href="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/toc.yml" rel="noopener noreferrer"><img src="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/toc.yml/badge.svg" alt="Generate TOC" /></a></h1><blockquote>
<p>Bioinformatics is an interdisciplinary field that develops methods and software tools for understanding biological data. — <a href="https://en.wikipedia.org/wiki/Bioinformatics" rel="noopener noreferrer">Wikipedia</a></p>
</blockquote>
<p>A curated list of awesome Bioinformatics software, resources, and libraries. Mostly command line based, and free or open-source. Please feel free to <a href="https://github.com/danielecook/Awesome-Bioinformatics/blob/master/README.md/CONTRIBUTING.md" rel="noopener noreferrer">contribute</a>!</p>




<h2 id="table-of-contents"><a class="anchor" aria-hidden="true" tabindex="-1" href="#table-of-contents"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Table of Contents</h2><ul>
<li><a href="#package-suites">Package suites</a></li>
<li><a href="#data-tools">Data Tools</a><ul>
<li><a href="#downloading">Downloading</a></li>
<li><a href="#compressing">Compressing</a></li>
</ul>
</li>
<li><a href="#data-processing">Data Processing</a><ul>
<li><a href="#command-line-utilities">Command Line Utilities</a></li>
</ul>
</li>
<li><a href="#next-generation-sequencing">Next Generation Sequencing</a><ul>
<li><a href="#workflow-managers">Workflow Managers</a></li>
<li><a href="#pipelines">Pipelines</a></li>
<li><a href="#sequence-processing">Sequence Processing</a></li>
<li><a href="#data-analysis">Data Analysis</a></li>
<li><a href="#sequence-alignment">Sequence Alignment</a><ul>
<li><a href="#pairwise">Pairwise</a></li>
<li><a href="#multiple-sequence-alignment">Multiple Sequence Alignment</a></li>
<li><a href="#clustering">Clustering</a></li>
</ul>
</li>
<li><a href="#quantification">Quantification</a></li>
<li><a href="#variant-calling">Variant Calling</a><ul>
<li><a href="#structural-variant-callers">Structural variant callers</a></li>
</ul>
</li>
<li><a href="#bam-file-utilities">BAM File Utilities</a></li>
<li><a href="#vcf-file-utilities">VCF File Utilities</a></li>
<li><a href="#gff-bed-file-utilities">GFF BED File Utilities</a></li>
<li><a href="#variant-simulation">Variant Simulation</a></li>
<li><a href="#variant-predictionannotation">Variant Prediction/Annotation</a></li>
<li><a href="#python-modules">Python Modules</a><ul>
<li><a href="#data">Data</a></li>
<li><a href="#tools">Tools</a></li>
</ul>
</li>
<li><a href="#assembly">Assembly</a></li>
<li><a href="#annotation">Annotation</a></li>
</ul>
</li>
<li><a href="#long-read-sequencing">Long-read sequencing</a><ul>
<li><a href="#long-read-assembly">Long-read Assembly</a></li>
</ul>
</li>
<li><a href="#visualization">Visualization</a><ul>
<li><a href="#genome-browsers--gene-diagrams">Genome Browsers / Gene Diagrams</a></li>
<li><a href="#circos-related">Circos Related</a></li>
</ul>
</li>
<li><a href="#database-access">Database Access</a></li>
<li><a href="#resources">Resources</a><ul>
<li><a href="#becoming-a-bioinformatician">Becoming a Bioinformatician</a></li>
<li><a href="#bioinformatics-on-github">Bioinformatics on GitHub</a></li>
<li><a href="#sequencing">Sequencing</a></li>
<li><a href="#rna-seq">RNA-Seq</a></li>
<li><a href="#chip-seq">ChIP-Seq</a></li>
<li><a href="#youtube-channels-and-playlists">YouTube Channels and Playlists</a></li>
<li><a href="#blogs">Blogs</a></li>
<li><a href="#miscellaneous">Miscellaneous</a></li>
</ul>
</li>
<li><a href="#online-networking-groups">Online networking groups</a></li>
<li><a href="#license">License</a></li>
</ul>


<hr />
<h2 id="package-suites"><a class="anchor" aria-hidden="true" tabindex="-1" href="#package-suites"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Package suites</h2><p>Package suites gather software packages and installation tools for specific languages or platforms. We have some for bioinformatics software.</p>
<ul>
<li><p><strong><a href="https://github.com/bioperl/bioperl-live" rel="noopener noreferrer">Bioperl</a></strong> - International association of users &amp; developers of open source Perl tools for bioinformatics, genomics and life sciences. [ <a href="https://doi.org/10.1101%2Fgr.361602" rel="noopener noreferrer">paper-2002</a> | <a href="https://bioperl.org" rel="noopener noreferrer">web</a> ]</p>
</li>
<li><p><strong><a href="https://github.com/Bioconductor" rel="noopener noreferrer">Bioconductor</a></strong> - A plethora of tools for analysis and comprehension of high-throughput genomic data, including 1500+ software packages. [ <a href="https://link.springer.com/article/10.1186/gb-2004-5-10-r80" rel="noopener noreferrer">paper-2004</a> | <a href="https://www.bioconductor.org" rel="noopener noreferrer">web</a> ]</p>
</li>
<li><p><strong><a href="https://github.com/biopython/biopython" rel="noopener noreferrer">Biopython</a></strong> - Freely available tools for biological computing in Python, with included cookbook, packaging and thorough documentation. Part of the <a href="http://open-bio.org/" rel="noopener noreferrer">Open Bioinformatics Foundation</a>. Contains the very useful <a href="https://biopython.org/DIST/docs/api/Bio.Entrez-module.html" rel="noopener noreferrer">Entrez</a> package for API access to the NCBI databases. [ <a href="https://pubmed.ncbi.nlm.nih.gov/19304878" rel="noopener noreferrer">paper-2009</a> | <a href="https://biopython.org" rel="noopener noreferrer">web</a> ]</p>
</li>
<li><p><strong><a href="https://github.com/bioconda" rel="noopener noreferrer">Bioconda</a></strong> - A channel for the <a href="http://conda.pydata.org/docs/intro.html" rel="noopener noreferrer">conda package manager</a> specializing in bioinformatics software. Includes a repository with 3000+ ready-to-install (with <code>conda install</code>) bioinformatics packages. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29967506" rel="noopener noreferrer">paper-2018</a> | <a href="https://bioconda.github.io" rel="noopener noreferrer">web</a> ]</p>
</li>
<li><p><strong><a href="https://github.com/BioJulia" rel="noopener noreferrer">BioJulia</a></strong> - Bioinformatics and computational biology infastructure for the Julia programming language. [ <a href="https://biojulia.net" rel="noopener noreferrer">web</a> ]</p>
</li>
<li><p><strong><a href="https://github.com/rust-bio/rust-bio" rel="noopener noreferrer">Rust-Bio</a></strong> - Rust implementations of algorithms and data structures useful for bioinformatics. [ <a href="http://bioinformatics.oxfordjournals.org/content/early/2015/10/06/bioinformatics.btv573.short?rss=1" rel="noopener noreferrer">paper-2016</a> ]</p>
</li>
<li><p><strong><a href="https://github.com/seqan/seqan3" rel="noopener noreferrer">SeqAn</a></strong> - The modern C++ library for sequence analysis.</p>
</li>
<li><p><strong><a href="https://github.com/TimothyStiles/poly" rel="noopener noreferrer">(Poly)merase</a></strong> - A Go library and command line utility for engineering organisms.</p>
</li>
<li><p><strong><a href="https://github.com/biocaml/biocaml" rel="noopener noreferrer">Biocaml</a></strong> - Biocaml aims to be a high-performance user-friendly library for Bioinformatics.</p>
</li>
<li><p><strong><a href="https://github.com/biojava/biojava" rel="noopener noreferrer">Biojava</a></strong> - Java framework for processing biological data.</p>
</li>
</ul>
<h2 id="data-tools"><a class="anchor" aria-hidden="true" tabindex="-1" href="#data-tools"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Data Tools</h2><h3 id="downloading"><a class="anchor" aria-hidden="true" tabindex="-1" href="#downloading"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Downloading</h3><ul>
<li><strong><a href="https://github.com/gogetdata/ggd-cli" rel="noopener noreferrer">GGD</a></strong> - Go Get Data; A command line interface for obtaining genomic data. [ <a href="https://gogetdata.github.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/ewels/sra-explorer" rel="noopener noreferrer">SRA-Explorer</a></strong> - Easily get SRA download links and other information. [ <a href="https://sra-explorer.info" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="compressing"><a class="anchor" aria-hidden="true" tabindex="-1" href="#compressing"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Compressing</h3><ul>
<li><strong><a href="https://github.com/divonlan/genozip" rel="noopener noreferrer">Genozip</a></strong> - A compressor of common genomic file formats (BAM, CRAM, FASTQ, VCF etc). [ <a href="https://genozip.com/?utm_source=Awesome-Bioinformatics" rel="noopener noreferrer">web</a> | <a href="https://www.researchgate.net/publication/349347156_Genozip_-_A_Universal_Extensible_Genomic_Data_Compressor" rel="noopener noreferrer">paper-2021</a> ]</li>
</ul>
<h2 id="data-processing"><a class="anchor" aria-hidden="true" tabindex="-1" href="#data-processing"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Data Processing</h2><h3 id="command-line-utilities"><a class="anchor" aria-hidden="true" tabindex="-1" href="#command-line-utilities"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Command Line Utilities</h3><ul>
<li><strong><a href="https://github.com/stephenturner/oneliners" rel="noopener noreferrer">Bioinformatics One Liners</a></strong> - Git repo of useful single line commands.</li>
<li><strong><a href="https://github.com/bionode/bionode" rel="noopener noreferrer">BioNode</a></strong> - Modular and universal bioinformatics, Bionode provides pipeable UNIX command line tools and JavaScript APIs for bioinformatics analysis workflows. [ <a href="http://bionode.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/bioSyntax/bioSyntax" rel="noopener noreferrer">bioSyntax</a></strong> - Syntax Highlighting for Computational Biology file formats (SAM, VCF, GTF, FASTA, PDB, etc...) in vim/less/gedit/sublime. [ <a href="https://pubmed.ncbi.nlm.nih.gov/30134911" rel="noopener noreferrer">paper-2018</a> | <a href="http://www.bioSyntax.org" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/wireservice/csvkit" rel="noopener noreferrer">CSVKit</a></strong> - Utilities for working with CSV/Tab-delimited files. [ <a href="https://csvkit.readthedocs.io/en/latest" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/shenwei356/csvtk" rel="noopener noreferrer">csvtk</a></strong> - Another cross-platform, efficient, practical and pretty CSV/TSV toolkit. [ <a href="https://bioinf.shenwei.me/csvtk" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://git.savannah.gnu.org/gitweb/?p=datamash.git" rel="noopener noreferrer">datamash</a></strong> - Data transformations and statistics. [ <a href="http://www.gnu.org/software/datamash" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/shenwei356/easy_qsub" rel="noopener noreferrer">easy_qsub</a></strong> - Easily submitting PBS jobs with script template. Multiple input files supported.</li>
<li><strong>GNU Parallel</strong> - General parallelizer that runs jobs in parallel on a single multi-core machine. <a href="https://www.biostars.org/p/63816/" rel="noopener noreferrer">Here</a> are some example scripts using GNU Parallel. [ <a href="http://www.gnu.org/software/parallel" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/arq5x/grabix" rel="noopener noreferrer">grabix</a></strong> - A wee tool for random access into BGZF files.</li>
<li><strong><a href="https://github.com/brentp/gsort" rel="noopener noreferrer">gsort</a></strong> - Sort genomic files according to a specified order.</li>
<li><strong><a href="https://github.com/samtools/tabix" rel="noopener noreferrer">tabix</a></strong> - Table file index. [ <a href="https://pubmed.ncbi.nlm.nih.gov/21208982" rel="noopener noreferrer">paper-2011</a> ]</li>
<li><strong><a href="https://github.com/wormtable/wormtable" rel="noopener noreferrer">wormtable</a></strong> - Write-once-read-many table for large datasets.</li>
<li><strong><a href="https://github.com/mattgodbolt/zindex" rel="noopener noreferrer">zindex</a></strong> - Create an index on a compressed text file.</li>
</ul>
<h2 id="next-generation-sequencing"><a class="anchor" aria-hidden="true" tabindex="-1" href="#next-generation-sequencing"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Next Generation Sequencing</h2><h3 id="workflow-managers"><a class="anchor" aria-hidden="true" tabindex="-1" href="#workflow-managers"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Workflow Managers</h3><ul>
<li><strong><a href="https://github.com/pcingola/BigDataScript" rel="noopener noreferrer">BigDataScript</a></strong> - A cross-system scripting language for working with big data pipelines in computer systems of different sizes and capabilities. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25189778" rel="noopener noreferrer">paper-2014</a> | <a href="https://pcingola.github.io/BigDataScript" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/ssadedin/bpipe" rel="noopener noreferrer">Bpipe</a></strong> - A small language for defining pipeline stages and linking them together to make pipelines. [ <a href="http://docs.bpipe.org" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/common-workflow-language/common-workflow-language" rel="noopener noreferrer">Common Workflow Language</a></strong> - a specification for describing analysis workflows and tools that are portable and scalable across a variety of software and hardware environments, from workstations to cluster, cloud, and high performance computing (HPC) environments. [ <a href="http://www.commonwl.org" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/broadinstitute/cromwell" rel="noopener noreferrer">Cromwell</a></strong> - A Workflow Management System geared towards scientific workflows. [ <a href="https://cromwell.readthedocs.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/galaxyproject" rel="noopener noreferrer">Galaxy</a></strong> - a popular open-source, web-based platform for data intensive biomedical research. Has several features, from data analysis to workflow management to visualization tools. [ <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030816" rel="noopener noreferrer">paper-2018</a> | <a href="https://galaxyproject.org" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/nextflow-io/nextflow" rel="noopener noreferrer">Nextflow</a> (recommended)</strong> - A fluent DSL modelled around the UNIX pipe concept, that simplifies writing parallel and scalable pipelines in a portable manner. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29412134" rel="noopener noreferrer">paper-2018</a> | <a href="http://nextflow.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/insitro/redun" rel="noopener noreferrer">redun</a></strong> - A python-based workflow manager.</li>
<li><strong><a href="https://github.com/cgat-developers/ruffus" rel="noopener noreferrer">Ruffus</a></strong> - Computation Pipeline library for python widely used in science and bioinformatics. [ <a href="https://pubmed.ncbi.nlm.nih.gov/20847218" rel="noopener noreferrer">paper-2010</a> | <a href="http://www.ruffus.org.uk" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/scipipe/scipipe" rel="noopener noreferrer">SciPipe</a></strong> - Workflow library embedded in the Go programming language, focusing on supporting complex workflow constructs, compiling to a single binary, providing powerful file naming and comprehensive audit reports for every output [ <a href="https://pubmed.ncbi.nlm.nih.gov/31029061/" rel="noopener noreferrer">paper-2019</a> | <a href="https://scipipe.org/" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/SeqWare/seqware" rel="noopener noreferrer">SeqWare</a></strong> - Hadoop Oozie-based workflow system focused on genomics data analysis in cloud environments. [ <a href="https://pubmed.ncbi.nlm.nih.gov/21210981" rel="noopener noreferrer">paper-2010</a> | <a href="https://seqware.github.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://bitbucket.org/snakemake" rel="noopener noreferrer">Snakemake</a></strong> - A workflow management system in Python that aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29788404" rel="noopener noreferrer">paper-2018</a> | <a href="https://snakemake.readthedocs.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/broadinstitute/wdl" rel="noopener noreferrer">Workflow Descriptor Language</a></strong> - Workflow standard developed by the Broad. [ <a href="https://software.broadinstitute.org/wdl" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="pipelines"><a class="anchor" aria-hidden="true" tabindex="-1" href="#pipelines"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Pipelines</h3><ul>
<li><strong><a href="https://github.com/pditommaso/awesome-pipeline" rel="noopener noreferrer">Awesome-Pipeline</a></strong> - A list of pipeline resources.</li>
<li><strong><a href="https://github.com/bactopia/bactopia/" rel="noopener noreferrer">Bactopia</a></strong> - A flexible pipeline, built with Nextflow, for the complete analysis of bacterial genomes. [ <a href="https://bactopia.github.io/" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/fmalmeida/bacannot" rel="noopener noreferrer">Bacannot</a></strong> - A generic but comprehensive bacterial annotation pipeline, built with Nextflow, with nice graphical options for investigating results. [ <a href="https://bacannot.readthedocs.io/en/latest/?badge=latest" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/chapmanb/bcbio-nextgen" rel="noopener noreferrer">bcbio-nextgen</a></strong> - Batteries included genomic analysis pipeline for variant and RNA-Seq analysis, structural variant calling, annotation, and prediction. [ <a href="https://bcbio-nextgen.readthedocs.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/pentalpha/r-peridot" rel="noopener noreferrer">R-Peridot</a></strong> - Customizable pipeline for differential expression analysis with an intuitive GUI. [ <a href="http://www.bioinformatics-brazil.org/r-peridot" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/fmalmeida/ngs-preprocess" rel="noopener noreferrer">ngs-preprocess</a></strong> - A pipeline for preprocessing short and long sequencing reads, built with Nextflow. [ <a href="https://ngs-preprocess.readthedocs.io/en/latest/?badge=latest" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="sequence-processing"><a class="anchor" aria-hidden="true" tabindex="-1" href="#sequence-processing"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Sequence Processing</h3><p>Sequence Processing includes tasks such as demultiplexing raw read data, and trimming low quality bases.</p>
<ul>
<li><strong><a href="https://github.com/OpenGene/AfterQC" rel="noopener noreferrer">AfterQC</a></strong> - Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data. [ <a href="https://pubmed.ncbi.nlm.nih.gov/28361673" rel="noopener noreferrer">paper-2017</a> ]</li>
<li><strong><a href="https://github.com/s-andrews/FastQC" rel="noopener noreferrer">FastQC</a></strong> - A quality control tool for high throughput sequence data. [ <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/mdshw5/fastqp" rel="noopener noreferrer">Fastqp</a></strong> - FASTQ and SAM quality control using Python.</li>
<li><strong><a href="https://github.com/agordon/fastx_toolkit" rel="noopener noreferrer">Fastx Tookit</a></strong> - FASTQ/A short-reads pre-processing tools: Demultiplexing, trimming, clipping, quality filtering, and masking utilities. [ <a href="http://hannonlab.cshl.edu/fastx_toolkit" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/ewels/MultiQC" rel="noopener noreferrer">MultiQC</a></strong> - Aggregate results from bioinformatics analyses across many samples into a single report. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27312411" rel="noopener noreferrer">paper-2016</a> | <a href="http://multiqc.info" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/telatin/seqfu2" rel="noopener noreferrer">SeqFu</a></strong> - Sequence manipulation toolkit for FASTA/FASTQ files written in Nim. [ <a href="https://www.mdpi.com/2306-5354/8/5/59" rel="noopener noreferrer">paper-2021</a> | <a href="https://telatin.github.io/seqfu2/" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/shenwei356/seqkit" rel="noopener noreferrer">SeqKit</a></strong> - A cross-platform and ultrafast toolkit for FASTA/Q file manipulation in Golang. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27706213" rel="noopener noreferrer">paper-2016</a> | <a href="https://bioinf.shenwei.me/seqkit" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/fhcrc/seqmagick" rel="noopener noreferrer">seqmagick</a></strong> - file format conversion in Biopython in a convenient way. [ <a href="http://seqmagick.readthedocs.io" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/lh3/seqtk" rel="noopener noreferrer">Seqtk</a></strong> - Toolkit for processing sequences in FASTA/Q formats.</li>
<li><strong><a href="https://github.com/incertae-sedis/smof" rel="noopener noreferrer">smof</a></strong> - UNIX-style FASTA manipulation tools.</li>
</ul>
<h3 id="data-analysis"><a class="anchor" aria-hidden="true" tabindex="-1" href="#data-analysis"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Data Analysis</h3><p>The following items allow for scalable genomic analysis by introducing specialized databases.</p>
<ul>
<li><strong><a href="https://github.com/hail-is/hail" rel="noopener noreferrer">Hail</a></strong> - Scalable genomic analysis.</li>
<li><strong><a href="https://github.com/dnanexus-rnd/GLnexus" rel="noopener noreferrer">GLNexus</a></strong> - Scalable gVCF merging and joint variant calling for population sequencing projects. [ <a href="https://www.biorxiv.org/content/10.1101/343970v1.abstract" rel="noopener noreferrer">paper-2018</a> ]</li>
</ul>
<h3 id="sequence-alignment"><a class="anchor" aria-hidden="true" tabindex="-1" href="#sequence-alignment"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Sequence Alignment</h3><h4 id="pairwise"><a class="anchor" aria-hidden="true" tabindex="-1" href="#pairwise"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Pairwise</h4><ul>
<li><strong><a href="https://github.com/BenLangmead/bowtie2" rel="noopener noreferrer">Bowtie 2</a></strong> - An ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. [ <a href="https://pubmed.ncbi.nlm.nih.gov/22388286" rel="noopener noreferrer">paper-2012</a> | <a href="http://bowtie-bio.sourceforge.net/bowtie2" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/lh3/bwa" rel="noopener noreferrer">BWA</a></strong> - Burrow-Wheeler Aligner for pairwise alignment between DNA sequences.</li>
<li><strong><a href="https://github.com/smarco/WFA" rel="noopener noreferrer">WFA</a></strong> - the wavefront alignment algorithm (WFA) which expoit sequence similarity to speed up alignment [ <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa777/5904262" rel="noopener noreferrer">paper-2020</a> ]</li>
<li><strong><a href="https://github.com/jeffdaily/parasail" rel="noopener noreferrer">Parasail</a></strong> - SIMD C library for global, semi-global, and local pairwise sequence alignments [ <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0930-z" rel="noopener noreferrer">paper-2016</a> ]</li>
<li><strong><a href="https://github.com/mummer4/mummer" rel="noopener noreferrer">MUMmer</a></strong> -  A system for rapidly aligning entire genomes, whether in complete or draft form. [ <a href="http://mummer.sourceforge.net/MUMmer.pdf" rel="noopener noreferrer">paper-1999</a> | <a href="http://mummer.sourceforge.net/MUMmer2.pdf" rel="noopener noreferrer">paper-2002</a> | <a href="http://mummer.sourceforge.net/MUMmer3.pdf" rel="noopener noreferrer">paper-2004</a> | <a href="http://mummer.sourceforge.net" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/bbuchfink/diamond" rel="noopener noreferrer">DIAMOND</a></strong> - An ultrafast protein aligner for <code>blastp</code> and <code>blastx</code> like searches. [ <a href="https://www.nature.com/articles/s41592-021-01101-x" rel="noopener noreferrer">paper-2021</a> ]</li>
</ul>
<h4 id="multiple-sequence-alignment"><a class="anchor" aria-hidden="true" tabindex="-1" href="#multiple-sequence-alignment"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Multiple Sequence Alignment</h4><ul>
<li><strong><a href="https://github.com/ljdursi/poapy" rel="noopener noreferrer">POA</a></strong> - Partial-Order Alignment for fast alignment and consensus of multiple homologous sequences. [ <a href="https://academic.oup.com/bioinformatics/article/18/3/452/236691" rel="noopener noreferrer">paper-2002</a> ]</li>
</ul>
<h4 id="clustering"><a class="anchor" aria-hidden="true" tabindex="-1" href="#clustering"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Clustering</h4><ul>
<li><strong><a href="https://github.com/soedinglab/MMseqs2" rel="noopener noreferrer">MMseqs2</a></strong> - Ultra-fast, sensitive search and clustering suite for protein and nucleotide sequence sets. [ <a href="https://www.nature.com/articles/nbt.3988" rel="noopener noreferrer">paper-2017</a> | <a href="https://www.nature.com/articles/s41467-018-04964-5" rel="noopener noreferrer">paper-2018</a> ]</li>
</ul>
<h3 id="quantification"><a class="anchor" aria-hidden="true" tabindex="-1" href="#quantification"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Quantification</h3><ul>
<li><strong><a href="https://github.com/cole-trapnell-lab/cufflinks" rel="noopener noreferrer">Cufflinks</a></strong> - Cufflinks assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples. [ <a href="https://www.nature.com/articles/nbt.1621" rel="noopener noreferrer">paper-2010</a> ]</li>
<li><strong><a href="https://github.com/deweylab/RSEM" rel="noopener noreferrer">RSEM</a></strong> - A software package for estimating gene and isoform expression levels from RNA-Seq data. [ <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323" rel="noopener noreferrer">paper-2011</a> | <a href="http://deweylab.github.io/RSEM/" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="variant-calling"><a class="anchor" aria-hidden="true" tabindex="-1" href="#variant-calling"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Variant Calling</h3><ul>
<li><strong><a href="https://github.com/google/deepvariant" rel="noopener noreferrer">DeepVariant</a></strong> - Deep learning-based variant caller [ <a href="https://rdcu.be/7Dhl" rel="noopener noreferrer">paper-2018</a> ]</li>
<li><strong><a href="https://github.com/ekg/freebayes" rel="noopener noreferrer">freebayes</a></strong> - Bayesian haplotype-based polymorphism discovery and genotyping. [ <a href="http://arxiv.org/abs/1207.3907" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/broadgsa/gatk" rel="noopener noreferrer">GATK</a></strong> - Variant Discovery in High-Throughput Sequencing Data. [ <a href="https://software.broadinstitute.org/gatk" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/luntergroup/octopus" rel="noopener noreferrer">Octopus</a></strong> - A polymorphic bayesian genotyping model with wide applicability. [ <a href="https://www.nature.com/articles/s41587-021-00861-3" rel="noopener noreferrer">paper-2021</a> ]</li>
<li><strong><a href="https://github.com/samtools/bcftools" rel="noopener noreferrer">bcftools</a></strong> - samtools/bcftools are a suite of tools for manipulating NGS data and can be used to call variants. [ <a href="https://pubmed.ncbi.nlm.nih.gov/19505943" rel="noopener noreferrer">paper-2009</a> | <a href="http://htslib.org" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h4 id="structural-variant-callers"><a class="anchor" aria-hidden="true" tabindex="-1" href="#structural-variant-callers"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Structural variant callers</h4><ul>
<li><strong><a href="https://github.com/dellytools/delly" rel="noopener noreferrer">Delly</a></strong> - Structural variant discovery by integrated paired-end and split-read analysis. [ <a href="https://pubmed.ncbi.nlm.nih.gov/22962449" rel="noopener noreferrer">paper-2012</a> ]</li>
<li><strong><a href="https://github.com/arq5x/lumpy-sv" rel="noopener noreferrer">lumpy</a></strong> - lumpy: a general probabilistic framework for structural variant discovery. [ <a href="https://link.springer.com/article/10.1186/gb-2014-15-6-r84" rel="noopener noreferrer">paper-2014</a> ]</li>
<li><strong><a href="https://github.com/Illumina/manta" rel="noopener noreferrer">manta</a></strong> - Structural variant and indel caller for mapped sequencing data. [ <a href="https://pubmed.ncbi.nlm.nih.gov/26647377" rel="noopener noreferrer">paper-2015</a> ]</li>
<li><strong><a href="https://github.com/PapenfussLab/gridss" rel="noopener noreferrer">gridss</a></strong> - GRIDSS: the Genomic Rearrangement IDentification Software Suite. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29097403" rel="noopener noreferrer">paper-2017</a> ]</li>
<li><strong><a href="https://github.com/brentp/smoove" rel="noopener noreferrer">smoove</a></strong> - structural variant calling and genotyping with existing tools, but,smoothly.</li>
</ul>
<h3 id="bam-file-utilities"><a class="anchor" aria-hidden="true" tabindex="-1" href="#bam-file-utilities"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>BAM File Utilities</h3><ul>
<li><strong><a href="https://github.com/pezmaster31/bamtools" rel="noopener noreferrer">Bamtools</a></strong> - Collection of tools for working with BAM files. [ <a href="https://academic.oup.com/bioinformatics/article/27/12/1691/255399" rel="noopener noreferrer">paper-2011</a> ]</li>
<li><strong><a href="https://github.com/AndersenLab/bam-toolbox" rel="noopener noreferrer">bam toolbox</a></strong> MtDNA:Nuclear Coverage; BAM Toolbox can output the ratio of MtDNA:nuclear coverage, a proxy for mitochondrial content.</li>
<li><strong><a href="https://github.com/DarwinAwardWinner/mergesam" rel="noopener noreferrer">mergesam</a></strong> - Automate common SAM &amp; BAM conversions.</li>
<li><strong><a href="https://github.com/brentp/mosdepth" rel="noopener noreferrer">mosdepth</a></strong> - fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29096012/" rel="noopener noreferrer">paper-2017</a> ]</li>
<li><strong><a href="https://github.com/TimoLassmann/samstat" rel="noopener noreferrer">SAMstat</a></strong> - Displaying sequence statistics for next-generation sequencing. [ <a href="https://academic.oup.com/bioinformatics/article/27/1/130/201972" rel="noopener noreferrer">paper-2010</a> | <a href="http://samstat.sourceforge.net" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/brentp/somalier" rel="noopener noreferrer">Somalier</a></strong> - Fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs. [ <a href="https://pubmed.ncbi.nlm.nih.gov/32664994" rel="noopener noreferrer">paper-2020</a> ]</li>
<li><strong><a href="https://github.com/zd1/telseq" rel="noopener noreferrer">Telseq</a></strong> - Telseq is a tool for estimating telomere length from whole genome sequence data. [ <a href="https://academic.oup.com/nar/article/42/9/e75/1249448" rel="noopener noreferrer">paper-2014</a> ]</li>
</ul>
<h3 id="vcf-file-utilities"><a class="anchor" aria-hidden="true" tabindex="-1" href="#vcf-file-utilities"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>VCF File Utilities</h3><ul>
<li><strong><a href="https://github.com/samtools/bcftools" rel="noopener noreferrer">bcftools</a></strong> - Set of tools for manipulating VCF files. [ <a href="https://pubmed.ncbi.nlm.nih.gov/26826718" rel="noopener noreferrer">paper-2016</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/28205675" rel="noopener noreferrer">paper-2017</a> | <a href="http://samtools.github.io/bcftools" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/brentp/vcfanno" rel="noopener noreferrer">vcfanno</a></strong> - Annotate a VCF with other VCFs/BEDs/tabixed files. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27250555" rel="noopener noreferrer">paper-2016</a> ]</li>
<li><strong><a href="https://github.com/vcflib/vcflib" rel="noopener noreferrer">vcflib</a></strong> - A C++ library for parsing and manipulating VCF files.</li>
<li><strong><a href="https://github.com/vcftools/vcftools" rel="noopener noreferrer">vcftools</a></strong> - VCF manipulation and statistics (e.g. linkage disequilibrium, allele frequency, Fst). [ <a href="https://pubmed.ncbi.nlm.nih.gov/21653522" rel="noopener noreferrer">paper-2011</a> ]</li>
</ul>
<h3 id="gff-bed-file-utilities"><a class="anchor" aria-hidden="true" tabindex="-1" href="#gff-bed-file-utilities"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>GFF BED File Utilities</h3><ul>
<li><strong><a href="https://github.com/NBISweden/AGAT" rel="noopener noreferrer">AGAT</a></strong> - Suite of tools to handle gene annotations in any GTF/GFF format. [ <a href="https://agat.readthedocs.io/en/latest/?badge=latest" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/daler/gffutils" rel="noopener noreferrer">gffutils</a></strong> - GFF and GTF file manipulation and interconversion. [ <a href="http://daler.github.io/gffutils" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://bedops.readthedocs.io/en/latest/index.html" rel="noopener noreferrer">BEDOPS</a></strong> - The fast, highly scalable and easily-parallelizable genome analysis toolkit. [ <a href="https://academic.oup.com/bioinformatics/article/28/14/1919/218826" rel="noopener noreferrer">paper-2012</a> ]</li>
<li><strong><a href="https://github.com/arq5x/bedtools2" rel="noopener noreferrer">Bedtools2</a></strong> - A Swiss Army knife for genome arithmetic. [ <a href="https://pubmed.ncbi.nlm.nih.gov/20110278" rel="noopener noreferrer">paper-2010</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/25199790" rel="noopener noreferrer">paper-2014</a> | <a href="https://bedtools.readthedocs.io" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="variant-simulation"><a class="anchor" aria-hidden="true" tabindex="-1" href="#variant-simulation"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Variant Simulation</h3><ul>
<li><strong><a href="https://github.com/adamewing/bamsurgeon" rel="noopener noreferrer">Bam Surgeon</a></strong> - Tools for adding mutations to existing <code>.bam</code> files, used for testing mutation callers. [ <a href="https://popmodels.cancercontrol.cancer.gov/gsr/packages/bamsurgeon" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/lh3/wgsim" rel="noopener noreferrer">wgsim</a></strong> - <strong>Comes with samtools!</strong> - Reads simulator. [ <a href="https://popmodels.cancercontrol.cancer.gov/gsr/packages/wgsim" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="variant-predictionannotation"><a class="anchor" aria-hidden="true" tabindex="-1" href="#variant-predictionannotation"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Variant Prediction/Annotation</h3><ul>
<li><strong><a href="https://github.com/teamdfir/sift" rel="noopener noreferrer">SIFT</a></strong> - Predicts whether an amino acid substitution affects protein function. [ <a href="https://pubmed.ncbi.nlm.nih.gov/12824425" rel="noopener noreferrer">paper-2003</a> | <a href="http://sift.jcvi.org" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/pcingola/SnpEff" rel="noopener noreferrer">SnpEff</a></strong> - Genetic variant annotation and effect prediction toolbox. [ <a href="https://www.tandfonline.com/doi/full/10.4161/fly.19695" rel="noopener noreferrer">paper-2012</a> | <a href="https://pcingola.github.io/SnpEff" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://anaconda.org/bioconda/ensembl-vep" rel="noopener noreferrer">Ensembl VEP</a></strong> - The VEP determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. [ <a href="https://doi.org/10.1186/s13059-016-0974-4" rel="noopener noreferrer">paper-2016</a> | <a href="http://www.ensembl.org/info/docs/tools/vep/index.html" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="python-modules"><a class="anchor" aria-hidden="true" tabindex="-1" href="#python-modules"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Python Modules</h3><h4 id="data"><a class="anchor" aria-hidden="true" tabindex="-1" href="#data"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Data</h4><ul>
<li><strong><a href="https://github.com/brentp/cruzdb" rel="noopener noreferrer">cruzdb</a></strong> - Pythonic access to the UCSC Genome database. [ <a href="https://academic.oup.com/bioinformatics/article/29/23/3003/248468" rel="noopener noreferrer">paper-2013</a> ]</li>
<li><strong><a href="https://github.com/openvax/pyensembl" rel="noopener noreferrer">pyensembl</a></strong> - Pythonic Access to the Ensembl database. [ <a href="https://pyensembl.readthedocs.io/en/latest/pyensembl.html" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/cokelaer/bioservices" rel="noopener noreferrer">bioservices</a></strong> - Access to Biological Web Services from Python. [ <a href="https://academic.oup.com/bioinformatics/article/29/24/3241/194040" rel="noopener noreferrer">paper-2013</a> | <a href="http://bioservices.readthedocs.io" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h4 id="tools"><a class="anchor" aria-hidden="true" tabindex="-1" href="#tools"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Tools</h4><ul>
<li><strong><a href="https://github.com/arq5x/cyvcf" rel="noopener noreferrer">cyvcf</a></strong> - A port of <a href="https://github.com/jamescasbon/PyVCF" rel="noopener noreferrer">pyVCF</a> using Cython for speed.</li>
<li><strong><a href="https://github.com/brentp/cyvcf2" rel="noopener noreferrer">cyvcf2</a></strong> - Cython + HTSlib == fast VCF parsing; even faster parsing than pyVCF. [ <a href="https://pubmed.ncbi.nlm.nih.gov/28165109" rel="noopener noreferrer">paper-2017</a> | <a href="https://brentp.github.io/cyvcf2" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/daler/pybedtools" rel="noopener noreferrer">pyBedTools</a></strong> - Python wrapper for <a href="https://github.com/arq5x/bedtools" rel="noopener noreferrer">bedtools</a>. [ <a href="https://pubmed.ncbi.nlm.nih.gov/21949271" rel="noopener noreferrer">paper-2011</a> | <a href="http://daler.github.io/pybedtools" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/mdshw5/pyfaidx" rel="noopener noreferrer">pyfaidx</a></strong> - Pythonic access to FASTA files.</li>
<li><strong><a href="https://github.com/pysam-developers/pysam" rel="noopener noreferrer">pysam</a></strong> - Python wrapper for <a href="https://github.com/samtools/samtools" rel="noopener noreferrer">samtools</a>. [ <a href="https://pysam.readthedocs.io/en/latest/api.html" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/jamescasbon/PyVCF" rel="noopener noreferrer">pyVCF</a></strong> - A VCF Parser for Python. [ <a href="http://pyvcf.readthedocs.org/en/latest/index.html" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="assembly"><a class="anchor" aria-hidden="true" tabindex="-1" href="#assembly"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Assembly</h3><ul>
<li><strong><a href="https://github.com/ablab/spades" rel="noopener noreferrer">SPAdes</a></strong> - SPAdes (St. Petersburg genome assembler) is an assembly toolkit containing various assembly pipelines and the de-facto standard for prokaryotic genome assemblies.</li>
<li><strong><a href="https://github.com/ncbi/SKESA" rel="noopener noreferrer">SKESA</a></strong> - SKESA is a de-novo sequence read assembler for microbial genomes. It uses conservative heuristics and is designed to create breaks at repeat regions in the genome. This leads to excellent sequence quality without significantly compromising contiguity.</li>
<li><strong><a href="https://github.com/lh3/minimap2" rel="noopener noreferrer">Minimap2</a></strong> - Minimap2 is an pairwise aligner for genomic and spliced nucleotide sequences. It can perform the assembly-to-assembly alignment, and works with gzip'd FASTQ, FASTA formats. It also finds overlaps between long-reads.</li>
<li><strong><a href="https://dgenies.toulouse.inra.fr/" rel="noopener noreferrer">D-GENIES</a></strong> - <strong>D</strong>ot plot large <strong>Gen</strong>omes in an <strong>I</strong>nteractive, <strong>E</strong>fficient and <strong>S</strong>imple way. It is an online tool designed to support large genome, compare two genomes, and you can interact with the dot plot to improve the visualisation. It can also be used for extension of minimap2 by uploading the output generated in PAF(Pairwise mApping Format) or MAF(Multiple Alignment File) alignment files to D-GENIES</li>
</ul>
<h3 id="annotation"><a class="anchor" aria-hidden="true" tabindex="-1" href="#annotation"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Annotation</h3><ul>
<li><strong><a href="https://github.com/tseemann/prokka" rel="noopener noreferrer">Prokka</a></strong> - Prokka: rapid prokaryotic genome annotation. Prokka is one of the most cited annotation command line tools for microbial genome annotations.</li>
<li><strong><a href="https://github.com/oschwengers/bakta" rel="noopener noreferrer">Bakta</a></strong> - Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes &amp; plasmids. It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis.</li>
</ul>
<h2 id="long-read-sequencing"><a class="anchor" aria-hidden="true" tabindex="-1" href="#long-read-sequencing"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Long-read sequencing</h2><h3 id="long-read-assembly"><a class="anchor" aria-hidden="true" tabindex="-1" href="#long-read-assembly"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Long-read Assembly</h3><ul>
<li><strong><a href="https://github.com/marbl/canu" rel="noopener noreferrer">canu</a></strong> - A single molecule sequence assembler for genomes large and small.</li>
<li><strong><a href="https://github.com/fenderglass/Flye" rel="noopener noreferrer">flye</a></strong> - De novo assembler for single molecule sequencing reads using repeat graphs.</li>
<li><strong><a href="https://github.com/chhylp123/hifiasm" rel="noopener noreferrer">hifiasm</a></strong> - A haplotype-resolved assembler for accurate Hifi reads.</li>
<li><strong><a href="https://github.com/ruanjue/wtdbg2" rel="noopener noreferrer">wtdbg2</a></strong> -  A fuzzy Bruijn graph approach to long noisy reads assembly</li>
</ul>
<h2 id="visualization"><a class="anchor" aria-hidden="true" tabindex="-1" href="#visualization"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Visualization</h2><h3 id="genome-browsers--gene-diagrams"><a class="anchor" aria-hidden="true" tabindex="-1" href="#genome-browsers--gene-diagrams"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Genome Browsers / Gene Diagrams</h3><p>The following tools can be used to visualize genomic data or for constructing customized visualizations of genomic data including sequence data from DNA-Seq, RNA-Seq, and ChIP-Seq, variants, and more.</p>
<ul>
<li><strong><a href="https://github.com/Lab41/squiggle" rel="noopener noreferrer">Squiggle</a></strong> - Easy-to-use DNA sequence visualization tool that turns FASTA files into browser-based visualizations. [ <a href="https://pubmed.ncbi.nlm.nih.gov/30247632" rel="noopener noreferrer">paper-2018</a> | <a href="https://squiggle.readthedocs.io/en/latest/" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/dasmoth/dalliance" rel="noopener noreferrer">biodalliance</a></strong> - Embeddable genome viewer. Integration data from a wide variety of sources, and can load data directly from popular genomics file formats including bigWig, BAM, and VCF.
[ <a href="https://pubmed.ncbi.nlm.nih.gov/21252075" rel="noopener noreferrer">paper-2011</a> | <a href="http://www.biodalliance.org" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/biojs/biojs" rel="noopener noreferrer">BioJS</a></strong> - BioJS is a library of over hundred JavaScript components enabling you to visualize and process data using current web technologies. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25075290/" rel="noopener noreferrer">paper-2014</a> | <a href="http://biojs.net/" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/jonathancrabtree/Circleator" rel="noopener noreferrer">Circleator</a></strong> - Flexible circular visualization of genome-associated data with BioPerl and SVG. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25075113" rel="noopener noreferrer">paper-2014</a> ]</li>
<li><strong><a href="https://github.com/drio/dnaism" rel="noopener noreferrer">DNAism</a></strong> - Horizon chart D3-based JavaScript library for DNA data. [ <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0891-2" rel="noopener noreferrer">paper-2016</a> | <a href="http://drio.github.io/dnaism/" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/igvteam/igv" rel="noopener noreferrer">IGV js</a></strong> - Java-based browser. Fast, efficient, scalable visualization tool for genomics data and annotations. Handles a large variety of formats. [ <a href="https://pubmed.ncbi.nlm.nih.gov/31099383" rel="noopener noreferrer">paper-2019</a> | <a href="https://software.broadinstitute.org/software/igv" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/lairdm/islandplot" rel="noopener noreferrer">Island Plot</a></strong> - D3 JavaScript based genome viewer. Constructs SVGs. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25916842/" rel="noopener noreferrer">paper-2015</a> ]</li>
<li><strong><a href="https://github.com/GMOD/jbrowse" rel="noopener noreferrer">JBrowse</a></strong> - JavaScript genome browser that is highly customizable via plugins and track customizations. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27072794" rel="noopener noreferrer">paper-2016</a> | <a href="http://jbrowse.org/" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/chgibb/PHAT" rel="noopener noreferrer">PHAT</a></strong> - Point and click, cross platform suite for analysing and visualizing next-generation sequencing datasets. [ <a href="https://pubmed.ncbi.nlm.nih.gov/30561651" rel="noopener noreferrer">paper-2018</a> | <a href="https://chgibb.github.io/PHATDocs" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/hammerlab/pileup.js" rel="noopener noreferrer">pileup.js</a></strong> - JavaScript library that can be used to generate interactive and highly customizable web-based genome browsers. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27153605" rel="noopener noreferrer">paper-2016</a> ]</li>
<li><strong><a href="https://github.com/chmille4/Scribl" rel="noopener noreferrer">scribl</a></strong> - JavaScript library for drawing canvas-based gene diagrams. [ <a href="https://pubmed.ncbi.nlm.nih.gov/23172864" rel="noopener noreferrer">paper-2012</a> | <a href="http://chmille4.github.io/Scribl" rel="noopener noreferrer">web</a> ]</li>
<li><strong>Lucid Align</strong> - A modern sequence alignment viewer. [ <a href="https://lucidalign.com" rel="noopener noreferrer">web</a> ]</li>
</ul>
<h3 id="circos-related"><a class="anchor" aria-hidden="true" tabindex="-1" href="#circos-related"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Circos Related</h3><ul>
<li><strong><a href="https://github.com/vigsterkr/circos" rel="noopener noreferrer">Circos</a></strong> - Perl package for circular plots, which are well suited for genomic rearrangements. [ <a href="https://pubmed.ncbi.nlm.nih.gov/19541911" rel="noopener noreferrer">paper-2009</a> | <a href="http://circos.ca" rel="noopener noreferrer">web</a> ]</li>
<li><strong>ClicO FS</strong> - An interactive web-based service of Circos. [ <a href="https://pubmed.ncbi.nlm.nih.gov/26227146" rel="noopener noreferrer">paper-2015</a> ]</li>
<li><strong>OmicCircos</strong> - R package for circular plots for omics data. [ <a href="https://pubmed.ncbi.nlm.nih.gov/24526832" rel="noopener noreferrer">paper-2014</a> | <a href="http://www.bioconductor.org/packages/release/bioc/html/OmicCircos.html" rel="noopener noreferrer">web</a> ]</li>
<li><strong>J-Circos</strong> - A Java application for doing interactive work with circos plots. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25540184" rel="noopener noreferrer">paper-2014</a> | <a href="http://www.australianprostatecentre.org/research/software/jcircos" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://bitbucket.org/henryhzhang/rcircos/src/master/" rel="noopener noreferrer">rCircos</a></strong> - R package for circular plots. [ <a href="https://pubmed.ncbi.nlm.nih.gov/23937229" rel="noopener noreferrer">paper-2013</a> | <a href="http://watson.nci.nih.gov/cran_mirror/web/packages/RCircos/index.html" rel="noopener noreferrer">web</a> ]</li>
<li><strong><a href="https://github.com/mkanai/fujiplot" rel="noopener noreferrer">fujiplot</a></strong> - A circos representation of multiple GWAS results. [ <a href="https://www.nature.com/articles/s41588-018-0047-6" rel="noopener noreferrer">paper-2018</a> ]</li>
</ul>
<h2 id="database-access"><a class="anchor" aria-hidden="true" tabindex="-1" href="#database-access"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Database Access</h2><ul>
<li><a href="http://www.ncbi.nlm.nih.gov/books/NBK179288/" rel="noopener noreferrer">Entrez Direct: E-utilities on the UNIX command line</a> - UNIX command line tools to access NCBI's databases programmatically. Instructions to install and examples are found in the link.</li>
</ul>
<h2 id="resources"><a class="anchor" aria-hidden="true" tabindex="-1" href="#resources"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Resources</h2><h3 id="becoming-a-bioinformatician"><a class="anchor" aria-hidden="true" tabindex="-1" href="#becoming-a-bioinformatician"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Becoming a Bioinformatician</h3><ul>
<li><a href="http://blog.fejes.ca/?p=2418" rel="noopener noreferrer">What is a bioinformatician</a></li>
<li><a href="http://www.ploscompbiol.org/article/info:doi%2F10.1371%2Fjournal.pcbi.1003496" rel="noopener noreferrer">Bioinformatics Curriculum Guidelines: Toward a Definition of Core Competencies</a></li>
<li><a href="http://caseybergman.wordpress.com/2012/07/31/top-n-reasons-to-do-a-ph-d-or-post-doc-in-bioinformaticscomputational-biology/" rel="noopener noreferrer">Top N Reasons To Do A Ph.D. or Post-Doc in Bioinformatics/Computational Biology</a></li>
<li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-1-104" rel="noopener noreferrer">A 10-Step Guide to Party Conversation For Bioinformaticians</a> - Here is a step-by-step guide on how to convey concepts to people not involved in the field when asked the question: 'So, what do you do?'</li>
<li><a href="https://www.youtube.com/watch?v=uwsjwMO-TEA" rel="noopener noreferrer">A History Of Bioinformatics (In The Year 2039)</a> - A talk by C. Titus Brown on his take of looking back at bioinformatics from the year 2039. His notes for this talk can be found <a href="http://ivory.idyll.org/blog/2014-bosc-keynote.html" rel="noopener noreferrer">here</a>.</li>
<li><a href="https://madhadron.com/science/farewell_to_bioinformatics.html" rel="noopener noreferrer">A farewell to bioinformatics</a> - A critical view of the state of bioinformatics.</li>
<li><a href="http://www.acgt.me/blog/2014/3/25/101-questions-a-new-series-of-interviews-with-notable-bioinformaticians" rel="noopener noreferrer">A Series of Interviews with Notable Bioinformaticians</a> - Dr. Keith Bradnam "thought it might be instructive to ask a simple series of questions to a bunch of notable bioinformaticians to assess their feelings on the current state of bioinformatics research, and maybe get any tips they have about what has been useful to their bioinformatics careers."</li>
<li><a href="https://github.com/ossu/bioinformatics" rel="noopener noreferrer">Open Source Society University on Bioinformatics</a> - Solid path for those of you who want to complete a Bioinformatics course on your own time, for free, with courses from the best universities in the World.</li>
<li><a href="http://rosalind.info/" rel="noopener noreferrer">Rosalind</a> - Rosalind is a platform for learning bioinformatics through problem solving.</li>
<li><a href="http://www.opiniomics.org/a-guide-for-the-lonely-bioinformatician/" rel="noopener noreferrer">A guide for the lonely bioinformatician</a> - This guide is aimed at bioinformaticians, and is meant to guide them towards better career development.</li>
<li><a href="https://doi.org/10.1093/bib/bby063" rel="noopener noreferrer">A brief history of bioinformatics</a></li>
</ul>
<h3 id="bioinformatics-on-github"><a class="anchor" aria-hidden="true" tabindex="-1" href="#bioinformatics-on-github"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Bioinformatics on GitHub</h3><ul>
<li><a href="https://github.com/HussainAther/awesome-alternative-splicing" rel="noopener noreferrer">Awesome-alternative-splicing</a> - List of resources on alternative splicing including software, databases, and other tools.</li>
<li><a href="https://github.com/opendilab/awesome-AI-based-protein-design" rel="noopener noreferrer">Awesome AI-based Protein Design</a> - A collection of research papers for AI-based protein design.</li>
</ul>
<h3 id="sequencing"><a class="anchor" aria-hidden="true" tabindex="-1" href="#sequencing"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Sequencing</h3><ul>
<li><a href="https://youtu.be/6Is3W7JkFp8" rel="noopener noreferrer">Next-Generation Sequencing Technologies - Elaine Mardis (2014)</a> [1:34:35] - Excellent (technical) overview of next-generation and third-generation sequencing technologies, along with some applications in cancer research.</li>
<li><a href="https://liorpachter.wordpress.com/seq/" rel="noopener noreferrer">Annotated bibliography of *Seq assays</a> - List of ~100 papers on various sequencing technologies and assays ranging from transcription to transposable element discovery.</li>
<li><a href="http://www.illumina.com/content/dam/illumina-marketing/documents/applications/ngs-library-prep/ForAllYouSeqMethods.pdf" rel="noopener noreferrer">For all you seq... (PDF)</a> (3456x5471) - Massive infographic by Illumina on illustrating how many sequencing techniques work. Techniques cover protein-protein interactions, RNA transcription, RNA-protein interactions, RNA low-level detection, RNA modifications, RNA structure, DNA rearrangements and markers, DNA low-level detection, epigenetics, and DNA-protein interactions. References included.</li>
</ul>
<h3 id="rna-seq"><a class="anchor" aria-hidden="true" tabindex="-1" href="#rna-seq"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>RNA-Seq</h3><ul>
<li><a href="https://www.biostars.org/p/52152/" rel="noopener noreferrer">Review papers on RNA-seq (Biostars)</a> - Includes lots of seminal papers on RNA-seq and analysis methods.</li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial" rel="noopener noreferrer">Informatics for RNA-seq: A web resource for analysis on the cloud</a> - Educational resource on performing RNA-seq analysis in the cloud using Amazon AWS cloud services. Topics include preparing the data, preprocessing, differential expression, isoform discovery, data visualization, and interpretation.</li>
<li><a href="http://rnaseq.uoregon.edu/" rel="noopener noreferrer">RNA-seqlopedia</a> - RNA-seqlopedia provides an awesome overview of RNA-seq and of the choices necessary to carry out a successful RNA-seq experiment.</li>
<li><a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8" rel="noopener noreferrer">A survey of best practices for RNA-seq data analysis</a> - Gives awesome roadmap for RNA-seq computational analyses, including challenges/obstacles and things to look out for, but also how you might integrate RNA-seq data with other data types.</li>
<li><a href="https://www.youtube.com/watch?v=5NiFibnbE8o" rel="noopener noreferrer">Stories from the Supplement</a> [46:39] - Dr. Lior Pachter shares his stories from the supplement for well-known RNA-seq analysis software CuffDiff and <a href="http://cole-trapnell-lab.github.io/cufflinks/" rel="noopener noreferrer">Cufflinks</a> and explains some of their methodologies.</li>
<li><a href="https://en.wikipedia.org/wiki/List_of_RNA-Seq_bioinformatics_tools" rel="noopener noreferrer">List of RNA-seq Bioinformatics Tools</a> - Extensive list on Wikipedia of RNA-seq bioinformatics tools needed in analysis, ranging from all parts of an analysis pipeline from quality control, alignment, splice analysis, and visualizations.</li>
<li><a href="https://github.com/crazyhottommy/RNA-seq-analysis" rel="noopener noreferrer">RNA-seq Analysis</a> - <a href="https://github.com/crazyhottommy" rel="noopener noreferrer">@crazyhottommy</a>'s notes on various steps and considerations when doing RNA-seq analysis.</li>
</ul>
<h3 id="chip-seq"><a class="anchor" aria-hidden="true" tabindex="-1" href="#chip-seq"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>ChIP-Seq</h3><ul>
<li><a href="https://github.com/crazyhottommy/ChIP-seq-analysis" rel="noopener noreferrer">ChIP-seq analysis notes from Tommy Tang</a> - Resources on ChIP-seq data which include papers, methods, links to software, and analysis.</li>
</ul>
<h3 id="youtube-channels-and-playlists"><a class="anchor" aria-hidden="true" tabindex="-1" href="#youtube-channels-and-playlists"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>YouTube Channels and Playlists</h3><ul>
<li><a href="https://www.genome.gov/12514288/current-topics-in-genome-analysis-2016-course-syllabus-handouts-and-videos/" rel="noopener noreferrer">Current Topics in Genome Analysis 2016</a> - Excellent series of fourteen lectures given at NIH about current topics in genomics ranging from sequence analysis, to sequencing technologies, and even more translational topics such as genomic medicine.</li>
<li><a href="https://www.youtube.com/user/GenomeTV" rel="noopener noreferrer">GenomeTV</a> - "GenomeTV is NHGRI's collection of official video resources from lectures, to news documentaries, to full video collections of meetings that tackle the research, issues and clinical applications of genomic research."</li>
<li><a href="https://www.youtube.com/user/LeadingStrand" rel="noopener noreferrer">Leading Strand</a> - Keynote lectures from Cold Spring Harbor Laboratory (CSHL) Meetings. More on <a href="http://theleadingstrand.cshl.edu/" rel="noopener noreferrer">The Leading Strand</a>.</li>
<li><a href="https://www.youtube.com/playlist?list=PLqLDR0CTP9_pboZCk6gR9Zn4kW7h9XWJI" rel="noopener noreferrer">Genomics, Big Data and Medicine Seminar Series</a> - "Our seminars are dedicated to the critical intersection of GBM, delving into 'bleeding edge' technology and approaches that will deeply shape the future."</li>
<li><a href="https://www.youtube.com/user/RafalabChannel/videos" rel="noopener noreferrer">Rafael Irizarry's Channel</a> - Dr. Rafael Irizarry's lectures and academic talks on statistics for genomics.</li>
<li><a href="https://www.youtube.com/user/nihvcast" rel="noopener noreferrer">NIH VideoCasting and Podcasting</a> - "NIH VideoCast broadcasts seminars, conferences and meetings live to a world-wide audience over the Internet as a real-time streaming video." Not exclusively genomics and bioinformatics video but many great talks on domain specific use of bioinformatics and genomics.</li>
</ul>
<h3 id="blogs"><a class="anchor" aria-hidden="true" tabindex="-1" href="#blogs"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Blogs</h3><ul>
<li><a href="http://www.acgt.me/" rel="noopener noreferrer">ACGT</a> - Dr. Keith Bradnam writes about this "thoughts on biology, genomics, and the ongoing threat to humanity from the bogus use of bioinformatics acroynums."</li>
<li><a href="http://www.opiniomics.org/" rel="noopener noreferrer">Opiniomics</a> - Dr. Mick Watson write on bioinformatics, genomes, and biology.</li>
<li><a href="https://liorpachter.wordpress.com/" rel="noopener noreferrer">Bits of DNA</a> - Dr. Lior Pachter writes review and commentary on computational biology.</li>
<li><a href="http://www.michaeleisen.org/blog/" rel="noopener noreferrer">it is NOT junk</a> - Dr. Michael Eisen writes "a blog about genomes, DNA, evolution, open science, baseball and other important things"</li>
<li><a href="https://bioinfoperl.blogspot.com" rel="noopener noreferrer">#!/perl/bioinfo</a> - The Computational and Structural Biology group at EEAD-CSIC writes, in Spanish and English, about ideas and code for plant genomics, computational and structural biology problems.</li>
</ul>
<h3 id="miscellaneous"><a class="anchor" aria-hidden="true" tabindex="-1" href="#miscellaneous"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Miscellaneous</h3><ul>
<li><a href="https://github.com/jtleek/genomicspapers/" rel="noopener noreferrer">The Leek group guide to genomics papers</a> - Expertly curated genomics papers to get up to speed on genomics, RNA-seq, statistics (used in genomics), software development, and more.</li>
<li><a href="https://doi.org/10.1371/journal.pcbi.1003662" rel="noopener noreferrer">A New Online Computational Biology Curriculum</a> - "This article introduces a catalog of several hundred free video courses of potential interest to those wishing to expand their knowledge of bioinformatics and computational biology. The courses are organized into eleven subject areas modeled on university departments and are accompanied by commentary and career advice."</li>
<li><a href="http://www.foo.be/docs/tpj/issues/vol1_2/tpj0102-0001.html" rel="noopener noreferrer">How Perl Saved the Human Genome Project</a> - An anecdote by Lincoln D. Stein on the importance of the Perl programming language in the Human Genome Project.</li>
<li><a href="https://liacs.leidenuniv.nl/~hoogeboomhj/mcb/nature_primer.html" rel="noopener noreferrer">Educational Papers from Nature Biotechnology and PLoS Computational Biology</a> - Page of links to primers and short educational articles on various methods used in computational biology and bioinformatics.</li>
<li><a href="https://peerj.com/collections/45-bioinformatics-software/" rel="noopener noreferrer">The PeerJ Bioinformatics Software Tools Collection</a> - Collection of tools curated by Keith Crandall and Claus White, aimed at collating the most interesting, innovative, and relevant bioinformatics tools articles in PeerJ.</li>
</ul>
<h2 id="online-networking-groups"><a class="anchor" aria-hidden="true" tabindex="-1" href="#online-networking-groups"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Online networking groups</h2><ul>
<li><a href="https://discord.com/invite/3uxbPns" rel="noopener noreferrer">Bioinformatics (on Discord)</a> - a Discord server for general bioinformatics</li>
<li><a href="https://www.reddit.com/r/bioinformatics/comments/7ndwm1/rbioinformatics_slack_channel_and_an_open_call/" rel="noopener noreferrer">r-bioinformatics</a> - the official Slack workspace of r/bioinformatics (<a href="https://www.reddit.com/message/compose/?to=apfejes&amp;subject=Request%20to%20join%20the%20r/bioinformatics%20Slack%20group&amp;message=I%20would%20like%20to%20request%20to%20join%20the%20r/bioinformatics%20Slack%20group" rel="noopener noreferrer">send a direct message to apfejes on reddit</a>)</li>
<li><a href="https://bioinformaticsgrx.es/" rel="noopener noreferrer">BioinformaticsGRX</a> - A community of bioinformaticians based in Granada, Spain</li>
<li><a href="https://comunidadbioinfo.github.io/" rel="noopener noreferrer">Comunidad de Desarolladores de Software en Bioinformática</a> - A community of bioinformaticians centered in Latin America</li>
<li><a href="https://combine.org.au/" rel="noopener noreferrer">COMBINE</a> - An Austrialian group for bioinformatics students</li>
</ul>
<h2 id="license"><a class="anchor" aria-hidden="true" tabindex="-1" href="#license"><svg class="octicon octicon-link" viewbox="0 0 16 16" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>License</h2><p><a href="https://creativecommons.org/publicdomain/zero/1.0/" rel="noopener noreferrer"><img src="http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg" alt="CC0" /></a></p>


    </main>
  </body>
</html>
